{
 "cells": [
  {
   "cell_type": "raw",
   "metadata": {},
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Preparation for folder obtained from Massey Genome Services (MGS)\n",
    "Get link and password (note password does not expire until you inform MGS that you have uploaded and verified data.\n",
    "Download your zip file and the mdsum.txt file using downloadthemall addon in Firefox. Transfer your file over to correct folder, for now use workspace/hrrmat/QIIME2... .\n",
    "\n",
    "Eric Burgueno: Download all data into Powerplant directly. Copy the following link in to Explorer Tab:  \\\\storage.powerplant.pfr.co.nz.  \n",
    "Copy them into “upload” share\\\\storage.powerplant.pfr.co.nz.\\input\\powerPlant\\incoming\\upload\\hrrmat .\n",
    "\n",
    "Use WinSCP to move the files to where you want them, and it should be very quick. \n",
    "However, only Bioinformatics has write access there so contact them to Store the sequencing data files under /storage.powerplant.pfr.co.nz/input/genomic/insect/Apis/ \n",
    "\n",
    "To make this trial work,fsequences were copied to /powerplant/workspace/Hrrmat\n",
    "To keep this tidy I would like them to be in /powerplant/workspace/Hrrmat/QIIME2/MGS6NAvsGA\n",
    "\n",
    "#Q2 workflow copied from https://github.com/PlantandFoodResearch/bioinf_Actinidia_metabarcoding/blob/master/QIIME2_v2-bigdata.ipynb"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 1. Describe your data location, name, and project name\n",
    "\n",
    "Please edit the following:\n",
    "\n",
    "1. Add your PFR username\n",
    "2. Add a descriptive project name (this will become the directory name of your project within `/workspace/<your_username>`\n",
    "3. Add the location of your data. This will generally be a directory of FASTQ files somewhere within `/input`"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "# Please edit USER to use your own PFR username.\n",
    "USER=Hrrmat\n",
    "# Please edit PROJECT to use your own project name (no spaces or special characters like #*?>/\\)\n",
    "PROJECT=MGS7R1_5dManuka\n",
    "# Please edit INPUT to describe where the metabarcoding sequence data currently sits.\n",
    "INPUT=/powerplant/workspace/Hrrmat/QIIME2/NZGL1664NZSurvey/fQsequencesV4SurveyPowell\n",
    "\n",
    "# Set your modulerc file. This is in case you have never logged in via SSH and thus do not\n",
    "# have an automatically generated file that allows you to use modules.\n",
    "# echo \"#%Module \n",
    "# module load use.own \n",
    "# setenv TCLLIBPATH /software/OSutils/modules/lib\"\n",
    "# > /home/$USER/.modulerc"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "# Unload QIIME2 if it's already loaded (to avoid conda conflicts)\n",
    "# Set the directory structure for the analysis.\n",
    "\n",
    "WORKING=/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka\n",
    "RAW=$WORKING/000.raw\n",
    "FASTQC=$WORKING/010.fastqc\n",
    "MULTIQC=$WORKING/020.multiqc\n",
    "IMPORT=$WORKING/030.import\n",
    "FILTER=$WORKING/040.filter\n",
    "OTU=$WORKING/050.otu\n",
    "PHYLOGENY=$WORKING/060.phylogeny\n",
    "DIVERSITY=$WORKING/070.diversity\n",
    "TAXONOMY=$WORKING/080.taxonomy\n",
    "DIFF=$WORKING/090.differential_abundance\n",
    "BIOM_TABLE=$WORKING/100.biom_table\n",
    "TMP=$WORKING/tmp\n",
    "mkdir -p $WORKING\n",
    "mkdir -p $RAW\n",
    "mkdir -p $FASTQC\n",
    "mkdir -p $MULTIQC\n",
    "mkdir -p $IMPORT\n",
    "mkdir -p $FILTER\n",
    "mkdir -p $OTU\n",
    "mkdir -p $PHYLOGENY\n",
    "mkdir -p $DIVERSITY\n",
    "mkdir -p $TAXONOMY\n",
    "mkdir -p $DIFF\n",
    "mkdir -p $TMP\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/powerplant/workspace/Hrrmat\n"
     ]
    }
   ],
   "source": [
    "pwd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/100.biom_table\n"
     ]
    }
   ],
   "source": [
    "echo $BIOM_TABLE"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 61,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "total 228\n",
      "2 1BD1_S11_L001_R1_001.fastq.gz    2 5dIS6_S72_L001_R1_001.fastq.gz\n",
      "2 1BD1_S11_L001_R2_001.fastq.gz    2 5dIS6_S72_L001_R2_001.fastq.gz\n",
      "2 1BD2_S12_L001_R1_001.fastq.gz    2 5dIS7_S73_L001_R1_001.fastq.gz\n",
      "2 1BD2_S12_L001_R2_001.fastq.gz    2 5dIS7_S73_L001_R2_001.fastq.gz\n",
      "2 1BD3_S13_L001_R1_001.fastq.gz    2 5dIS8_S74_L001_R1_001.fastq.gz\n",
      "2 1BD3_S13_L001_R2_001.fastq.gz    2 5dIS8_S74_L001_R2_001.fastq.gz\n",
      "2 1BD4_S14_L001_R1_001.fastq.gz    2 5dIS9_S75_L001_R1_001.fastq.gz\n",
      "2 1BD4_S14_L001_R2_001.fastq.gz    2 5dIS9_S75_L001_R2_001.fastq.gz\n",
      "2 1BD5_S15_L001_R1_001.fastq.gz    2 5dMGO1_S51_L001_R1_001.fastq.gz\n",
      "2 1BD5_S15_L001_R2_001.fastq.gz    2 5dMGO1_S51_L001_R2_001.fastq.gz\n",
      "2 1BJ1_S1_L001_R1_001.fastq.gz\t   2 5dMGO2_S52_L001_R1_001.fastq.gz\n",
      "2 1BJ1_S1_L001_R2_001.fastq.gz\t   2 5dMGO2_S52_L001_R2_001.fastq.gz\n",
      "2 1BJ2_S2_L001_R1_001.fastq.gz\t   2 5dMGO3_S53_L001_R1_001.fastq.gz\n",
      "2 1BJ2_S2_L001_R2_001.fastq.gz\t   2 5dMGO3_S53_L001_R2_001.fastq.gz\n",
      "2 1BJ3_S3_L001_R1_001.fastq.gz\t   2 5dMGO4_S54_L001_R1_001.fastq.gz\n",
      "2 1BJ3_S3_L001_R2_001.fastq.gz\t   2 5dMGO4_S54_L001_R2_001.fastq.gz\n",
      "2 1BJ4_S4_L001_R1_001.fastq.gz\t   2 5dMGO6_S55_L001_R1_001.fastq.gz\n",
      "2 1BJ4_S4_L001_R2_001.fastq.gz\t   2 5dMGO6_S55_L001_R2_001.fastq.gz\n",
      "2 1BJ5_S5_L001_R1_001.fastq.gz\t   2 5dMGO7_S56_L001_R1_001.fastq.gz\n",
      "2 1BJ5_S5_L001_R2_001.fastq.gz\t   2 5dMGO7_S56_L001_R2_001.fastq.gz\n",
      "2 1BM1_S16_L001_R1_001.fastq.gz    2 5dMGO8_S57_L001_R1_001.fastq.gz\n",
      "2 1BM1_S16_L001_R2_001.fastq.gz    2 5dMGO8_S57_L001_R2_001.fastq.gz\n",
      "2 1BM2_S17_L001_R1_001.fastq.gz    2 5dMGO9_S58_L001_R1_001.fastq.gz\n",
      "2 1BM2_S17_L001_R2_001.fastq.gz    2 5dMGO9_S58_L001_R2_001.fastq.gz\n",
      "2 1BM3_S18_L001_R1_001.fastq.gz    2 5dMH15-1_S35_L001_R1_001.fastq.gz\n",
      "2 1BM3_S18_L001_R2_001.fastq.gz    2 5dMH15-1_S35_L001_R2_001.fastq.gz\n",
      "2 1BM4_S19_L001_R1_001.fastq.gz    2 5dMH15-2_S36_L001_R1_001.fastq.gz\n",
      "2 1BM4_S19_L001_R2_001.fastq.gz    2 5dMH15-2_S36_L001_R2_001.fastq.gz\n",
      "2 1BM5_S20_L001_R1_001.fastq.gz    2 5dMH15-3_S37_L001_R1_001.fastq.gz\n",
      "2 1BM5_S20_L001_R2_001.fastq.gz    2 5dMH15-3_S37_L001_R2_001.fastq.gz\n",
      "2 1BS1_S6_L001_R1_001.fastq.gz\t   2 5dMH15-4_S38_L001_R1_001.fastq.gz\n",
      "2 1BS1_S6_L001_R2_001.fastq.gz\t   2 5dMH15-4_S38_L001_R2_001.fastq.gz\n",
      "2 1BS2_S7_L001_R1_001.fastq.gz\t   2 5dMH15-6_S39_L001_R1_001.fastq.gz\n",
      "2 1BS2_S7_L001_R2_001.fastq.gz\t   2 5dMH15-6_S39_L001_R2_001.fastq.gz\n",
      "2 1BS3_S8_L001_R1_001.fastq.gz\t   2 5dMH15-7_S40_L001_R1_001.fastq.gz\n",
      "2 1BS3_S8_L001_R2_001.fastq.gz\t   2 5dMH15-7_S40_L001_R2_001.fastq.gz\n",
      "2 1BS4_S9_L001_R1_001.fastq.gz\t   2 5dMH15-8_S41_L001_R1_001.fastq.gz\n",
      "2 1BS4_S9_L001_R2_001.fastq.gz\t   2 5dMH15-8_S41_L001_R2_001.fastq.gz\n",
      "2 1BS5_S10_L001_R1_001.fastq.gz    2 5dMH15-9_S42_L001_R1_001.fastq.gz\n",
      "2 1BS5_S10_L001_R2_001.fastq.gz    2 5dMH15-9_S42_L001_R2_001.fastq.gz\n",
      "2 5dDHA1_S59_L001_R1_001.fastq.gz  2 5dMH17-1_S43_L001_R1_001.fastq.gz\n",
      "2 5dDHA1_S59_L001_R2_001.fastq.gz  2 5dMH17-1_S43_L001_R2_001.fastq.gz\n",
      "2 5dDHA2_S60_L001_R1_001.fastq.gz  2 5dMH17-2_S44_L001_R1_001.fastq.gz\n",
      "2 5dDHA2_S60_L001_R2_001.fastq.gz  2 5dMH17-2_S44_L001_R2_001.fastq.gz\n",
      "2 5dDHA3_S61_L001_R1_001.fastq.gz  2 5dMH17-3_S45_L001_R1_001.fastq.gz\n",
      "2 5dDHA3_S61_L001_R2_001.fastq.gz  2 5dMH17-3_S45_L001_R2_001.fastq.gz\n",
      "2 5dDHA4_S62_L001_R1_001.fastq.gz  2 5dMH17-4_S46_L001_R1_001.fastq.gz\n",
      "2 5dDHA4_S62_L001_R2_001.fastq.gz  2 5dMH17-4_S46_L001_R2_001.fastq.gz\n",
      "2 5dDHA5_S63_L001_R1_001.fastq.gz  2 5dMH17-6_S47_L001_R1_001.fastq.gz\n",
      "2 5dDHA5_S63_L001_R2_001.fastq.gz  2 5dMH17-6_S47_L001_R2_001.fastq.gz\n",
      "2 5dDHA6_S64_L001_R1_001.fastq.gz  2 5dMH17-7_S48_L001_R1_001.fastq.gz\n",
      "2 5dDHA6_S64_L001_R2_001.fastq.gz  2 5dMH17-7_S48_L001_R2_001.fastq.gz\n",
      "2 5dDHA7_S65_L001_R1_001.fastq.gz  2 5dMH17-8_S49_L001_R1_001.fastq.gz\n",
      "2 5dDHA7_S65_L001_R2_001.fastq.gz  2 5dMH17-8_S49_L001_R2_001.fastq.gz\n",
      "2 5dDHA8_S66_L001_R1_001.fastq.gz  2 5dMH17-9_S50_L001_R1_001.fastq.gz\n",
      "2 5dDHA8_S66_L001_R2_001.fastq.gz  2 5dMH17-9_S50_L001_R2_001.fastq.gz\n",
      "2 5dDHA9_S67_L001_R1_001.fastq.gz  2 5dSI1_S31_L001_R1_001.fastq.gz\n",
      "2 5dDHA9_S67_L001_R2_001.fastq.gz  2 5dSI1_S31_L001_R2_001.fastq.gz\n",
      "2 5dH1_S28_L001_R1_001.fastq.gz    2 5dSI2_S32_L001_R1_001.fastq.gz\n",
      "2 5dH1_S28_L001_R2_001.fastq.gz    2 5dSI2_S32_L001_R2_001.fastq.gz\n",
      "2 5dH2_S29_L001_R1_001.fastq.gz    2 5dSI3_S33_L001_R1_001.fastq.gz\n",
      "2 5dH2_S29_L001_R2_001.fastq.gz    2 5dSI3_S33_L001_R2_001.fastq.gz\n",
      "2 5dH3_S30_L001_R1_001.fastq.gz    2 5dSI4_S34_L001_R1_001.fastq.gz\n",
      "2 5dH3_S30_L001_R2_001.fastq.gz    2 5dSI4_S34_L001_R2_001.fastq.gz\n",
      "2 5dH4_S26_L001_R1_001.fastq.gz    2 5dSI5_S21_L001_R1_001.fastq.gz\n",
      "2 5dH4_S26_L001_R2_001.fastq.gz    2 5dSI5_S21_L001_R2_001.fastq.gz\n",
      "2 5dH5_S27_L001_R1_001.fastq.gz    2 5dSI6_S22_L001_R1_001.fastq.gz\n",
      "2 5dH5_S27_L001_R2_001.fastq.gz    2 5dSI6_S22_L001_R2_001.fastq.gz\n",
      "2 5dIS1_S68_L001_R1_001.fastq.gz   2 5dSI7_S23_L001_R1_001.fastq.gz\n",
      "2 5dIS1_S68_L001_R2_001.fastq.gz   2 5dSI7_S23_L001_R2_001.fastq.gz\n",
      "2 5dIS2_S69_L001_R1_001.fastq.gz   2 5dSI8_S24_L001_R1_001.fastq.gz\n",
      "2 5dIS2_S69_L001_R2_001.fastq.gz   2 5dSI8_S24_L001_R2_001.fastq.gz\n",
      "2 5dIS3_S70_L001_R1_001.fastq.gz   2 5dSI9_S25_L001_R1_001.fastq.gz\n",
      "2 5dIS3_S70_L001_R2_001.fastq.gz   2 5dSI9_S25_L001_R2_001.fastq.gz\n",
      "2 5dIS4_S71_L001_R1_001.fastq.gz   2 Undetermined_S0_L001_R1_001.fastq.gz\n",
      "2 5dIS4_S71_L001_R2_001.fastq.gz   2 Undetermined_S0_L001_R2_001.fastq.gz\n"
     ]
    }
   ],
   "source": [
    "ls -s $RAW"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 2. Prepare your raw data\n",
    "The preparation of raw data will be quite specific to the project. In this case the raw data is in a subdirectory of a zipped file. QIIME2 can deal with data that is not demultiplexed but in this case the data has already been demultiplexed. We will extract the relevant raw data to the \"RAW\" directory and ensure the files are read/writeable.\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 72,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "total 228\n",
      "2 1BD1_S11_L001_R1_001.fastq.gz    2 5dIS6_S72_L001_R1_001.fastq.gz\n",
      "2 1BD1_S11_L001_R2_001.fastq.gz    2 5dIS6_S72_L001_R2_001.fastq.gz\n",
      "2 1BD2_S12_L001_R1_001.fastq.gz    2 5dIS7_S73_L001_R1_001.fastq.gz\n",
      "2 1BD2_S12_L001_R2_001.fastq.gz    2 5dIS7_S73_L001_R2_001.fastq.gz\n",
      "2 1BD3_S13_L001_R1_001.fastq.gz    2 5dIS8_S74_L001_R1_001.fastq.gz\n",
      "2 1BD3_S13_L001_R2_001.fastq.gz    2 5dIS8_S74_L001_R2_001.fastq.gz\n",
      "2 1BD4_S14_L001_R1_001.fastq.gz    2 5dIS9_S75_L001_R1_001.fastq.gz\n",
      "2 1BD4_S14_L001_R2_001.fastq.gz    2 5dIS9_S75_L001_R2_001.fastq.gz\n",
      "2 1BD5_S15_L001_R1_001.fastq.gz    2 5dMGO1_S51_L001_R1_001.fastq.gz\n",
      "2 1BD5_S15_L001_R2_001.fastq.gz    2 5dMGO1_S51_L001_R2_001.fastq.gz\n",
      "2 1BJ1_S1_L001_R1_001.fastq.gz\t   2 5dMGO2_S52_L001_R1_001.fastq.gz\n",
      "2 1BJ1_S1_L001_R2_001.fastq.gz\t   2 5dMGO2_S52_L001_R2_001.fastq.gz\n",
      "2 1BJ2_S2_L001_R1_001.fastq.gz\t   2 5dMGO3_S53_L001_R1_001.fastq.gz\n",
      "2 1BJ2_S2_L001_R2_001.fastq.gz\t   2 5dMGO3_S53_L001_R2_001.fastq.gz\n",
      "2 1BJ3_S3_L001_R1_001.fastq.gz\t   2 5dMGO4_S54_L001_R1_001.fastq.gz\n",
      "2 1BJ3_S3_L001_R2_001.fastq.gz\t   2 5dMGO4_S54_L001_R2_001.fastq.gz\n",
      "2 1BJ4_S4_L001_R1_001.fastq.gz\t   2 5dMGO6_S55_L001_R1_001.fastq.gz\n",
      "2 1BJ4_S4_L001_R2_001.fastq.gz\t   2 5dMGO6_S55_L001_R2_001.fastq.gz\n",
      "2 1BJ5_S5_L001_R1_001.fastq.gz\t   2 5dMGO7_S56_L001_R1_001.fastq.gz\n",
      "2 1BJ5_S5_L001_R2_001.fastq.gz\t   2 5dMGO7_S56_L001_R2_001.fastq.gz\n",
      "2 1BM1_S16_L001_R1_001.fastq.gz    2 5dMGO8_S57_L001_R1_001.fastq.gz\n",
      "2 1BM1_S16_L001_R2_001.fastq.gz    2 5dMGO8_S57_L001_R2_001.fastq.gz\n",
      "2 1BM2_S17_L001_R1_001.fastq.gz    2 5dMGO9_S58_L001_R1_001.fastq.gz\n",
      "2 1BM2_S17_L001_R2_001.fastq.gz    2 5dMGO9_S58_L001_R2_001.fastq.gz\n",
      "2 1BM3_S18_L001_R1_001.fastq.gz    2 5dMH15-1_S35_L001_R1_001.fastq.gz\n",
      "2 1BM3_S18_L001_R2_001.fastq.gz    2 5dMH15-1_S35_L001_R2_001.fastq.gz\n",
      "2 1BM4_S19_L001_R1_001.fastq.gz    2 5dMH15-2_S36_L001_R1_001.fastq.gz\n",
      "2 1BM4_S19_L001_R2_001.fastq.gz    2 5dMH15-2_S36_L001_R2_001.fastq.gz\n",
      "2 1BM5_S20_L001_R1_001.fastq.gz    2 5dMH15-3_S37_L001_R1_001.fastq.gz\n",
      "2 1BM5_S20_L001_R2_001.fastq.gz    2 5dMH15-3_S37_L001_R2_001.fastq.gz\n",
      "2 1BS1_S6_L001_R1_001.fastq.gz\t   2 5dMH15-4_S38_L001_R1_001.fastq.gz\n",
      "2 1BS1_S6_L001_R2_001.fastq.gz\t   2 5dMH15-4_S38_L001_R2_001.fastq.gz\n",
      "2 1BS2_S7_L001_R1_001.fastq.gz\t   2 5dMH15-6_S39_L001_R1_001.fastq.gz\n",
      "2 1BS2_S7_L001_R2_001.fastq.gz\t   2 5dMH15-6_S39_L001_R2_001.fastq.gz\n",
      "2 1BS3_S8_L001_R1_001.fastq.gz\t   2 5dMH15-7_S40_L001_R1_001.fastq.gz\n",
      "2 1BS3_S8_L001_R2_001.fastq.gz\t   2 5dMH15-7_S40_L001_R2_001.fastq.gz\n",
      "2 1BS4_S9_L001_R1_001.fastq.gz\t   2 5dMH15-8_S41_L001_R1_001.fastq.gz\n",
      "2 1BS4_S9_L001_R2_001.fastq.gz\t   2 5dMH15-8_S41_L001_R2_001.fastq.gz\n",
      "2 1BS5_S10_L001_R1_001.fastq.gz    2 5dMH15-9_S42_L001_R1_001.fastq.gz\n",
      "2 1BS5_S10_L001_R2_001.fastq.gz    2 5dMH15-9_S42_L001_R2_001.fastq.gz\n",
      "2 5dDHA1_S59_L001_R1_001.fastq.gz  2 5dMH17-1_S43_L001_R1_001.fastq.gz\n",
      "2 5dDHA1_S59_L001_R2_001.fastq.gz  2 5dMH17-1_S43_L001_R2_001.fastq.gz\n",
      "2 5dDHA2_S60_L001_R1_001.fastq.gz  2 5dMH17-2_S44_L001_R1_001.fastq.gz\n",
      "2 5dDHA2_S60_L001_R2_001.fastq.gz  2 5dMH17-2_S44_L001_R2_001.fastq.gz\n",
      "2 5dDHA3_S61_L001_R1_001.fastq.gz  2 5dMH17-3_S45_L001_R1_001.fastq.gz\n",
      "2 5dDHA3_S61_L001_R2_001.fastq.gz  2 5dMH17-3_S45_L001_R2_001.fastq.gz\n",
      "2 5dDHA4_S62_L001_R1_001.fastq.gz  2 5dMH17-4_S46_L001_R1_001.fastq.gz\n",
      "2 5dDHA4_S62_L001_R2_001.fastq.gz  2 5dMH17-4_S46_L001_R2_001.fastq.gz\n",
      "2 5dDHA5_S63_L001_R1_001.fastq.gz  2 5dMH17-6_S47_L001_R1_001.fastq.gz\n",
      "2 5dDHA5_S63_L001_R2_001.fastq.gz  2 5dMH17-6_S47_L001_R2_001.fastq.gz\n",
      "2 5dDHA6_S64_L001_R1_001.fastq.gz  2 5dMH17-7_S48_L001_R1_001.fastq.gz\n",
      "2 5dDHA6_S64_L001_R2_001.fastq.gz  2 5dMH17-7_S48_L001_R2_001.fastq.gz\n",
      "2 5dDHA7_S65_L001_R1_001.fastq.gz  2 5dMH17-8_S49_L001_R1_001.fastq.gz\n",
      "2 5dDHA7_S65_L001_R2_001.fastq.gz  2 5dMH17-8_S49_L001_R2_001.fastq.gz\n",
      "2 5dDHA8_S66_L001_R1_001.fastq.gz  2 5dMH17-9_S50_L001_R1_001.fastq.gz\n",
      "2 5dDHA8_S66_L001_R2_001.fastq.gz  2 5dMH17-9_S50_L001_R2_001.fastq.gz\n",
      "2 5dDHA9_S67_L001_R1_001.fastq.gz  2 5dSI1_S31_L001_R1_001.fastq.gz\n",
      "2 5dDHA9_S67_L001_R2_001.fastq.gz  2 5dSI1_S31_L001_R2_001.fastq.gz\n",
      "2 5dH1_S28_L001_R1_001.fastq.gz    2 5dSI2_S32_L001_R1_001.fastq.gz\n",
      "2 5dH1_S28_L001_R2_001.fastq.gz    2 5dSI2_S32_L001_R2_001.fastq.gz\n",
      "2 5dH2_S29_L001_R1_001.fastq.gz    2 5dSI3_S33_L001_R1_001.fastq.gz\n",
      "2 5dH2_S29_L001_R2_001.fastq.gz    2 5dSI3_S33_L001_R2_001.fastq.gz\n",
      "2 5dH3_S30_L001_R1_001.fastq.gz    2 5dSI4_S34_L001_R1_001.fastq.gz\n",
      "2 5dH3_S30_L001_R2_001.fastq.gz    2 5dSI4_S34_L001_R2_001.fastq.gz\n",
      "2 5dH4_S26_L001_R1_001.fastq.gz    2 5dSI5_S21_L001_R1_001.fastq.gz\n",
      "2 5dH4_S26_L001_R2_001.fastq.gz    2 5dSI5_S21_L001_R2_001.fastq.gz\n",
      "2 5dH5_S27_L001_R1_001.fastq.gz    2 5dSI6_S22_L001_R1_001.fastq.gz\n",
      "2 5dH5_S27_L001_R2_001.fastq.gz    2 5dSI6_S22_L001_R2_001.fastq.gz\n",
      "2 5dIS1_S68_L001_R1_001.fastq.gz   2 5dSI7_S23_L001_R1_001.fastq.gz\n",
      "2 5dIS1_S68_L001_R2_001.fastq.gz   2 5dSI7_S23_L001_R2_001.fastq.gz\n",
      "2 5dIS2_S69_L001_R1_001.fastq.gz   2 5dSI8_S24_L001_R1_001.fastq.gz\n",
      "2 5dIS2_S69_L001_R2_001.fastq.gz   2 5dSI8_S24_L001_R2_001.fastq.gz\n",
      "2 5dIS3_S70_L001_R1_001.fastq.gz   2 5dSI9_S25_L001_R1_001.fastq.gz\n",
      "2 5dIS3_S70_L001_R2_001.fastq.gz   2 5dSI9_S25_L001_R2_001.fastq.gz\n",
      "2 5dIS4_S71_L001_R1_001.fastq.gz   2 Undetermined_S0_L001_R1_001.fastq.gz\n",
      "2 5dIS4_S71_L001_R2_001.fastq.gz   2 Undetermined_S0_L001_R2_001.fastq.gz\n"
     ]
    }
   ],
   "source": [
    "# This step depends on your particular data.\n",
    "# You should create symlinks from your $INPUT fastq.gz files to $RAW\n",
    "ln -s $INPUT/*.fastq.gz $RAW;\n",
    "ls -s $RAW\n",
    "# Symlinks should be small (2kb) in size"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 3.Quality check of raw data\n",
    "We will run FastQC and MultiQC on the raw data to assess the quality. Assessments of quality are important for downstream steps where we decide on trimming parameters.\n",
    "This is seen in the Seasonal Carbohydrate MultiQC so does not need to be replicated."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Started analysis of 1BD1_S11_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 1BD1_S11_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 1BD1_S11_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 1BD1_S11_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 1BD1_S11_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 1BD1_S11_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 1BD1_S11_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 1BD1_S11_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 1BD1_S11_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 1BD1_S11_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 1BD1_S11_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 1BD1_S11_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 1BD1_S11_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 1BD1_S11_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 1BD1_S11_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 1BD1_S11_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 1BD1_S11_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 1BD1_S11_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 1BD1_S11_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 1BD1_S11_L001_R1_001.fastq.gz\n",
      "Analysis complete for 1BD1_S11_L001_R1_001.fastq.gz\n",
      "Started analysis of 1BD1_S11_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 1BD1_S11_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 1BD1_S11_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 1BD1_S11_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 1BD1_S11_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 1BD1_S11_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 1BD1_S11_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 1BD1_S11_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 1BD1_S11_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 1BD1_S11_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 1BD1_S11_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 1BD1_S11_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 1BD1_S11_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 1BD1_S11_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 1BD1_S11_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 1BD1_S11_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 1BD1_S11_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 1BD1_S11_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 1BD1_S11_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 1BD1_S11_L001_R2_001.fastq.gz\n",
      "Analysis complete for 1BD1_S11_L001_R2_001.fastq.gz\n",
      "Started analysis of 1BD2_S12_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 1BD2_S12_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 1BD2_S12_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 1BD2_S12_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 1BD2_S12_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 1BD2_S12_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 1BD2_S12_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 1BD2_S12_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 1BD2_S12_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 1BD2_S12_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 1BD2_S12_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 1BD2_S12_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 1BD2_S12_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 1BD2_S12_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 1BD2_S12_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 1BD2_S12_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 1BD2_S12_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 1BD2_S12_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 1BD2_S12_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 1BD2_S12_L001_R1_001.fastq.gz\n",
      "Analysis complete for 1BD2_S12_L001_R1_001.fastq.gz\n",
      "Started analysis of 1BD2_S12_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 1BD2_S12_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 1BD2_S12_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 1BD2_S12_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 1BD2_S12_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 1BD2_S12_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 1BD2_S12_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 1BD2_S12_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 1BD2_S12_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 1BD2_S12_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 1BD2_S12_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 1BD2_S12_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 1BD2_S12_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 1BD2_S12_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 1BD2_S12_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 1BD2_S12_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 1BD2_S12_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 1BD2_S12_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 1BD2_S12_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 1BD2_S12_L001_R2_001.fastq.gz\n",
      "Analysis complete for 1BD2_S12_L001_R2_001.fastq.gz\n",
      "Started analysis of 1BD3_S13_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 1BD3_S13_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 1BD3_S13_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 1BD3_S13_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 1BD3_S13_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 1BD3_S13_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 1BD3_S13_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 1BD3_S13_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 1BD3_S13_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 1BD3_S13_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 1BD3_S13_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 1BD3_S13_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 1BD3_S13_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 1BD3_S13_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 1BD3_S13_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 1BD3_S13_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 1BD3_S13_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 1BD3_S13_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 1BD3_S13_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 1BD3_S13_L001_R1_001.fastq.gz\n",
      "Analysis complete for 1BD3_S13_L001_R1_001.fastq.gz\n",
      "Started analysis of 1BD3_S13_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 1BD3_S13_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 1BD3_S13_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 1BD3_S13_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 1BD3_S13_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 1BD3_S13_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 1BD3_S13_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 1BD3_S13_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 1BD3_S13_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 1BD3_S13_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 1BD3_S13_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 1BD3_S13_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 1BD3_S13_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 1BD3_S13_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 1BD3_S13_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 1BD3_S13_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 1BD3_S13_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 1BD3_S13_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 1BD3_S13_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 1BD3_S13_L001_R2_001.fastq.gz\n",
      "Analysis complete for 1BD3_S13_L001_R2_001.fastq.gz\n",
      "Started analysis of 1BD4_S14_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 1BD4_S14_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 1BD4_S14_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 1BD4_S14_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 1BD4_S14_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 1BD4_S14_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 1BD4_S14_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 1BD4_S14_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 1BD4_S14_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 1BD4_S14_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 1BD4_S14_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 1BD4_S14_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 1BD4_S14_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 1BD4_S14_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 1BD4_S14_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 1BD4_S14_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 1BD4_S14_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 1BD4_S14_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 1BD4_S14_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 1BD4_S14_L001_R1_001.fastq.gz\n",
      "Analysis complete for 1BD4_S14_L001_R1_001.fastq.gz\n",
      "Started analysis of 1BD4_S14_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 1BD4_S14_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 1BD4_S14_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 1BD4_S14_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 1BD4_S14_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 1BD4_S14_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 30% complete for 1BD4_S14_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 1BD4_S14_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 1BD4_S14_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 1BD4_S14_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 1BD4_S14_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 1BD4_S14_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 1BD4_S14_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 1BD4_S14_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 1BD4_S14_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 1BD4_S14_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 1BD4_S14_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 1BD4_S14_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 1BD4_S14_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 1BD4_S14_L001_R2_001.fastq.gz\n",
      "Analysis complete for 1BD4_S14_L001_R2_001.fastq.gz\n",
      "Started analysis of 1BD5_S15_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 1BD5_S15_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 1BD5_S15_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 1BD5_S15_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 1BD5_S15_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 1BD5_S15_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 1BD5_S15_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 1BD5_S15_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 1BD5_S15_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 1BD5_S15_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 1BD5_S15_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 1BD5_S15_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 1BD5_S15_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 1BD5_S15_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 1BD5_S15_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 1BD5_S15_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 1BD5_S15_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 1BD5_S15_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 1BD5_S15_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 1BD5_S15_L001_R1_001.fastq.gz\n",
      "Analysis complete for 1BD5_S15_L001_R1_001.fastq.gz\n",
      "Started analysis of 1BD5_S15_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 1BD5_S15_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 1BD5_S15_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 1BD5_S15_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 1BD5_S15_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 1BD5_S15_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 1BD5_S15_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 1BD5_S15_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 1BD5_S15_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 1BD5_S15_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 1BD5_S15_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 1BD5_S15_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 1BD5_S15_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 1BD5_S15_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 1BD5_S15_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 1BD5_S15_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 1BD5_S15_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 1BD5_S15_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 1BD5_S15_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 1BD5_S15_L001_R2_001.fastq.gz\n",
      "Analysis complete for 1BD5_S15_L001_R2_001.fastq.gz\n",
      "Started analysis of 1BJ1_S1_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 1BJ1_S1_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 1BJ1_S1_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 1BJ1_S1_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 1BJ1_S1_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 1BJ1_S1_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 1BJ1_S1_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 1BJ1_S1_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 1BJ1_S1_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 1BJ1_S1_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 1BJ1_S1_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 1BJ1_S1_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 1BJ1_S1_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 1BJ1_S1_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 1BJ1_S1_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 1BJ1_S1_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 1BJ1_S1_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 1BJ1_S1_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 1BJ1_S1_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 1BJ1_S1_L001_R1_001.fastq.gz\n",
      "Analysis complete for 1BJ1_S1_L001_R1_001.fastq.gz\n",
      "Started analysis of 1BJ1_S1_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 1BJ1_S1_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 1BJ1_S1_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 1BJ1_S1_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 1BJ1_S1_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 1BJ1_S1_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 1BJ1_S1_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 1BJ1_S1_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 1BJ1_S1_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 1BJ1_S1_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 1BJ1_S1_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 1BJ1_S1_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 1BJ1_S1_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 1BJ1_S1_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 1BJ1_S1_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 1BJ1_S1_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 1BJ1_S1_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 1BJ1_S1_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 1BJ1_S1_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 1BJ1_S1_L001_R2_001.fastq.gz\n",
      "Analysis complete for 1BJ1_S1_L001_R2_001.fastq.gz\n",
      "Started analysis of 1BJ2_S2_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 1BJ2_S2_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 1BJ2_S2_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 1BJ2_S2_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 1BJ2_S2_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 1BJ2_S2_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 1BJ2_S2_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 1BJ2_S2_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 1BJ2_S2_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 1BJ2_S2_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 1BJ2_S2_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 1BJ2_S2_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 1BJ2_S2_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 1BJ2_S2_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 1BJ2_S2_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 1BJ2_S2_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 1BJ2_S2_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 1BJ2_S2_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 1BJ2_S2_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 1BJ2_S2_L001_R1_001.fastq.gz\n",
      "Analysis complete for 1BJ2_S2_L001_R1_001.fastq.gz\n",
      "Started analysis of 1BJ2_S2_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 1BJ2_S2_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 1BJ2_S2_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 1BJ2_S2_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 1BJ2_S2_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 1BJ2_S2_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 1BJ2_S2_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 1BJ2_S2_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 1BJ2_S2_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 1BJ2_S2_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 1BJ2_S2_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 1BJ2_S2_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 1BJ2_S2_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 1BJ2_S2_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 1BJ2_S2_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 1BJ2_S2_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 1BJ2_S2_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 1BJ2_S2_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 1BJ2_S2_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 1BJ2_S2_L001_R2_001.fastq.gz\n",
      "Analysis complete for 1BJ2_S2_L001_R2_001.fastq.gz\n",
      "Started analysis of 1BJ3_S3_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 1BJ3_S3_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 1BJ3_S3_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 1BJ3_S3_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 1BJ3_S3_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 1BJ3_S3_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 1BJ3_S3_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 1BJ3_S3_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 1BJ3_S3_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 1BJ3_S3_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 1BJ3_S3_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 1BJ3_S3_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 1BJ3_S3_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 1BJ3_S3_L001_R1_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 70% complete for 1BJ3_S3_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 1BJ3_S3_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 1BJ3_S3_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 1BJ3_S3_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 1BJ3_S3_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 1BJ3_S3_L001_R1_001.fastq.gz\n",
      "Analysis complete for 1BJ3_S3_L001_R1_001.fastq.gz\n",
      "Started analysis of 1BJ3_S3_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 1BJ3_S3_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 1BJ3_S3_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 1BJ3_S3_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 1BJ3_S3_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 1BJ3_S3_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 1BJ3_S3_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 1BJ3_S3_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 1BJ3_S3_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 1BJ3_S3_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 1BJ3_S3_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 1BJ3_S3_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 1BJ3_S3_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 1BJ3_S3_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 1BJ3_S3_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 1BJ3_S3_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 1BJ3_S3_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 1BJ3_S3_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 1BJ3_S3_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 1BJ3_S3_L001_R2_001.fastq.gz\n",
      "Analysis complete for 1BJ3_S3_L001_R2_001.fastq.gz\n",
      "Started analysis of 1BJ4_S4_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 1BJ4_S4_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 1BJ4_S4_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 1BJ4_S4_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 1BJ4_S4_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 1BJ4_S4_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 1BJ4_S4_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 1BJ4_S4_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 1BJ4_S4_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 1BJ4_S4_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 1BJ4_S4_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 1BJ4_S4_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 1BJ4_S4_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 1BJ4_S4_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 1BJ4_S4_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 1BJ4_S4_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 1BJ4_S4_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 1BJ4_S4_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 1BJ4_S4_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 1BJ4_S4_L001_R1_001.fastq.gz\n",
      "Analysis complete for 1BJ4_S4_L001_R1_001.fastq.gz\n",
      "Started analysis of 1BJ4_S4_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 1BJ4_S4_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 1BJ4_S4_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 1BJ4_S4_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 1BJ4_S4_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 1BJ4_S4_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 1BJ4_S4_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 1BJ4_S4_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 1BJ4_S4_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 1BJ4_S4_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 1BJ4_S4_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 1BJ4_S4_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 1BJ4_S4_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 1BJ4_S4_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 1BJ4_S4_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 1BJ4_S4_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 1BJ4_S4_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 1BJ4_S4_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 1BJ4_S4_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 1BJ4_S4_L001_R2_001.fastq.gz\n",
      "Analysis complete for 1BJ4_S4_L001_R2_001.fastq.gz\n",
      "Started analysis of 1BJ5_S5_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 1BJ5_S5_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 1BJ5_S5_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 1BJ5_S5_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 1BJ5_S5_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 1BJ5_S5_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 1BJ5_S5_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 1BJ5_S5_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 1BJ5_S5_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 1BJ5_S5_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 1BJ5_S5_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 1BJ5_S5_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 1BJ5_S5_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 1BJ5_S5_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 1BJ5_S5_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 1BJ5_S5_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 1BJ5_S5_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 1BJ5_S5_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 1BJ5_S5_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 1BJ5_S5_L001_R1_001.fastq.gz\n",
      "Analysis complete for 1BJ5_S5_L001_R1_001.fastq.gz\n",
      "Started analysis of 1BJ5_S5_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 1BJ5_S5_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 1BJ5_S5_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 1BJ5_S5_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 1BJ5_S5_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 1BJ5_S5_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 1BJ5_S5_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 1BJ5_S5_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 1BJ5_S5_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 1BJ5_S5_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 1BJ5_S5_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 1BJ5_S5_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 1BJ5_S5_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 1BJ5_S5_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 1BJ5_S5_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 1BJ5_S5_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 1BJ5_S5_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 1BJ5_S5_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 1BJ5_S5_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 1BJ5_S5_L001_R2_001.fastq.gz\n",
      "Analysis complete for 1BJ5_S5_L001_R2_001.fastq.gz\n",
      "Started analysis of 1BM1_S16_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 1BM1_S16_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 1BM1_S16_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 1BM1_S16_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 1BM1_S16_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 1BM1_S16_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 1BM1_S16_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 1BM1_S16_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 1BM1_S16_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 1BM1_S16_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 1BM1_S16_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 1BM1_S16_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 1BM1_S16_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 1BM1_S16_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 1BM1_S16_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 1BM1_S16_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 1BM1_S16_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 1BM1_S16_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 1BM1_S16_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 1BM1_S16_L001_R1_001.fastq.gz\n",
      "Analysis complete for 1BM1_S16_L001_R1_001.fastq.gz\n",
      "Started analysis of 1BM1_S16_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 1BM1_S16_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 1BM1_S16_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 1BM1_S16_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 1BM1_S16_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 1BM1_S16_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 1BM1_S16_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 1BM1_S16_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 1BM1_S16_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 1BM1_S16_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 1BM1_S16_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 1BM1_S16_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 1BM1_S16_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 1BM1_S16_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 1BM1_S16_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 1BM1_S16_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 1BM1_S16_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 1BM1_S16_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 1BM1_S16_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 1BM1_S16_L001_R2_001.fastq.gz\n",
      "Analysis complete for 1BM1_S16_L001_R2_001.fastq.gz\n",
      "Started analysis of 1BM2_S17_L001_R1_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 5% complete for 1BM2_S17_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 1BM2_S17_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 1BM2_S17_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 1BM2_S17_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 1BM2_S17_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 1BM2_S17_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 1BM2_S17_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 1BM2_S17_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 1BM2_S17_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 1BM2_S17_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 1BM2_S17_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 1BM2_S17_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 1BM2_S17_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 1BM2_S17_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 1BM2_S17_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 1BM2_S17_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 1BM2_S17_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 1BM2_S17_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 1BM2_S17_L001_R1_001.fastq.gz\n",
      "Analysis complete for 1BM2_S17_L001_R1_001.fastq.gz\n",
      "Started analysis of 1BM2_S17_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 1BM2_S17_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 1BM2_S17_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 1BM2_S17_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 1BM2_S17_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 1BM2_S17_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 1BM2_S17_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 1BM2_S17_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 1BM2_S17_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 1BM2_S17_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 1BM2_S17_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 1BM2_S17_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 1BM2_S17_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 1BM2_S17_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 1BM2_S17_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 1BM2_S17_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 1BM2_S17_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 1BM2_S17_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 1BM2_S17_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 1BM2_S17_L001_R2_001.fastq.gz\n",
      "Analysis complete for 1BM2_S17_L001_R2_001.fastq.gz\n",
      "Started analysis of 1BM3_S18_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 1BM3_S18_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 1BM3_S18_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 1BM3_S18_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 1BM3_S18_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 1BM3_S18_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 1BM3_S18_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 1BM3_S18_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 1BM3_S18_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 1BM3_S18_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 1BM3_S18_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 1BM3_S18_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 1BM3_S18_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 1BM3_S18_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 1BM3_S18_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 1BM3_S18_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 1BM3_S18_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 1BM3_S18_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 1BM3_S18_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 1BM3_S18_L001_R1_001.fastq.gz\n",
      "Analysis complete for 1BM3_S18_L001_R1_001.fastq.gz\n",
      "Started analysis of 1BM3_S18_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 1BM3_S18_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 1BM3_S18_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 1BM3_S18_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 1BM3_S18_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 1BM3_S18_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 1BM3_S18_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 1BM3_S18_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 1BM3_S18_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 1BM3_S18_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 1BM3_S18_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 1BM3_S18_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 1BM3_S18_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 1BM3_S18_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 1BM3_S18_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 1BM3_S18_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 1BM3_S18_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 1BM3_S18_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 1BM3_S18_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 1BM3_S18_L001_R2_001.fastq.gz\n",
      "Analysis complete for 1BM3_S18_L001_R2_001.fastq.gz\n",
      "Started analysis of 1BM4_S19_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 1BM4_S19_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 1BM4_S19_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 1BM4_S19_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 1BM4_S19_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 1BM4_S19_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 1BM4_S19_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 1BM4_S19_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 1BM4_S19_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 1BM4_S19_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 1BM4_S19_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 1BM4_S19_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 1BM4_S19_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 1BM4_S19_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 1BM4_S19_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 1BM4_S19_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 1BM4_S19_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 1BM4_S19_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 1BM4_S19_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 1BM4_S19_L001_R1_001.fastq.gz\n",
      "Analysis complete for 1BM4_S19_L001_R1_001.fastq.gz\n",
      "Started analysis of 1BM4_S19_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 1BM4_S19_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 1BM4_S19_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 1BM4_S19_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 1BM4_S19_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 1BM4_S19_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 1BM4_S19_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 1BM4_S19_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 1BM4_S19_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 1BM4_S19_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 1BM4_S19_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 1BM4_S19_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 1BM4_S19_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 1BM4_S19_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 1BM4_S19_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 1BM4_S19_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 1BM4_S19_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 1BM4_S19_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 1BM4_S19_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 1BM4_S19_L001_R2_001.fastq.gz\n",
      "Analysis complete for 1BM4_S19_L001_R2_001.fastq.gz\n",
      "Started analysis of 1BM5_S20_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 1BM5_S20_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 1BM5_S20_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 1BM5_S20_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 1BM5_S20_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 1BM5_S20_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 1BM5_S20_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 1BM5_S20_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 1BM5_S20_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 1BM5_S20_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 1BM5_S20_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 1BM5_S20_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 1BM5_S20_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 1BM5_S20_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 1BM5_S20_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 1BM5_S20_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 1BM5_S20_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 1BM5_S20_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 1BM5_S20_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 1BM5_S20_L001_R1_001.fastq.gz\n",
      "Analysis complete for 1BM5_S20_L001_R1_001.fastq.gz\n",
      "Started analysis of 1BM5_S20_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 1BM5_S20_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 1BM5_S20_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 1BM5_S20_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 1BM5_S20_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 1BM5_S20_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 1BM5_S20_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 35% complete for 1BM5_S20_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 1BM5_S20_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 1BM5_S20_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 1BM5_S20_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 1BM5_S20_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 1BM5_S20_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 1BM5_S20_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 1BM5_S20_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 1BM5_S20_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 1BM5_S20_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 1BM5_S20_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 1BM5_S20_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 1BM5_S20_L001_R2_001.fastq.gz\n",
      "Analysis complete for 1BM5_S20_L001_R2_001.fastq.gz\n",
      "Started analysis of 1BS1_S6_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 1BS1_S6_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 1BS1_S6_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 1BS1_S6_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 1BS1_S6_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 1BS1_S6_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 1BS1_S6_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 1BS1_S6_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 1BS1_S6_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 1BS1_S6_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 1BS1_S6_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 1BS1_S6_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 1BS1_S6_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 1BS1_S6_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 1BS1_S6_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 1BS1_S6_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 1BS1_S6_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 1BS1_S6_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 1BS1_S6_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 1BS1_S6_L001_R1_001.fastq.gz\n",
      "Analysis complete for 1BS1_S6_L001_R1_001.fastq.gz\n",
      "Started analysis of 1BS1_S6_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 1BS1_S6_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 1BS1_S6_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 1BS1_S6_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 1BS1_S6_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 1BS1_S6_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 1BS1_S6_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 1BS1_S6_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 1BS1_S6_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 1BS1_S6_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 1BS1_S6_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 1BS1_S6_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 1BS1_S6_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 1BS1_S6_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 1BS1_S6_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 1BS1_S6_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 1BS1_S6_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 1BS1_S6_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 1BS1_S6_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 1BS1_S6_L001_R2_001.fastq.gz\n",
      "Analysis complete for 1BS1_S6_L001_R2_001.fastq.gz\n",
      "Started analysis of 1BS2_S7_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 1BS2_S7_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 1BS2_S7_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 1BS2_S7_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 1BS2_S7_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 1BS2_S7_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 1BS2_S7_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 1BS2_S7_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 1BS2_S7_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 1BS2_S7_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 1BS2_S7_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 1BS2_S7_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 1BS2_S7_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 1BS2_S7_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 1BS2_S7_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 1BS2_S7_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 1BS2_S7_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 1BS2_S7_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 1BS2_S7_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 1BS2_S7_L001_R1_001.fastq.gz\n",
      "Analysis complete for 1BS2_S7_L001_R1_001.fastq.gz\n",
      "Started analysis of 1BS2_S7_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 1BS2_S7_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 1BS2_S7_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 1BS2_S7_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 1BS2_S7_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 1BS2_S7_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 1BS2_S7_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 1BS2_S7_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 1BS2_S7_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 1BS2_S7_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 1BS2_S7_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 1BS2_S7_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 1BS2_S7_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 1BS2_S7_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 1BS2_S7_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 1BS2_S7_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 1BS2_S7_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 1BS2_S7_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 1BS2_S7_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 1BS2_S7_L001_R2_001.fastq.gz\n",
      "Analysis complete for 1BS2_S7_L001_R2_001.fastq.gz\n",
      "Started analysis of 1BS3_S8_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 1BS3_S8_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 1BS3_S8_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 1BS3_S8_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 1BS3_S8_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 1BS3_S8_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 1BS3_S8_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 1BS3_S8_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 1BS3_S8_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 1BS3_S8_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 1BS3_S8_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 1BS3_S8_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 1BS3_S8_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 1BS3_S8_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 1BS3_S8_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 1BS3_S8_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 1BS3_S8_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 1BS3_S8_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 1BS3_S8_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 1BS3_S8_L001_R1_001.fastq.gz\n",
      "Analysis complete for 1BS3_S8_L001_R1_001.fastq.gz\n",
      "Started analysis of 1BS3_S8_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 1BS3_S8_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 1BS3_S8_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 1BS3_S8_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 1BS3_S8_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 1BS3_S8_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 1BS3_S8_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 1BS3_S8_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 1BS3_S8_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 1BS3_S8_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 1BS3_S8_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 1BS3_S8_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 1BS3_S8_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 1BS3_S8_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 1BS3_S8_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 1BS3_S8_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 1BS3_S8_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 1BS3_S8_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 1BS3_S8_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 1BS3_S8_L001_R2_001.fastq.gz\n",
      "Analysis complete for 1BS3_S8_L001_R2_001.fastq.gz\n",
      "Started analysis of 1BS4_S9_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 1BS4_S9_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 1BS4_S9_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 1BS4_S9_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 1BS4_S9_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 1BS4_S9_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 1BS4_S9_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 1BS4_S9_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 1BS4_S9_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 1BS4_S9_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 1BS4_S9_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 1BS4_S9_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 1BS4_S9_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 1BS4_S9_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 1BS4_S9_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 1BS4_S9_L001_R1_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 80% complete for 1BS4_S9_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 1BS4_S9_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 1BS4_S9_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 1BS4_S9_L001_R1_001.fastq.gz\n",
      "Analysis complete for 1BS4_S9_L001_R1_001.fastq.gz\n",
      "Started analysis of 1BS4_S9_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 1BS4_S9_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 1BS4_S9_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 1BS4_S9_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 1BS4_S9_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 1BS4_S9_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 1BS4_S9_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 1BS4_S9_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 1BS4_S9_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 1BS4_S9_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 1BS4_S9_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 1BS4_S9_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 1BS4_S9_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 1BS4_S9_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 1BS4_S9_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 1BS4_S9_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 1BS4_S9_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 1BS4_S9_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 1BS4_S9_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 1BS4_S9_L001_R2_001.fastq.gz\n",
      "Analysis complete for 1BS4_S9_L001_R2_001.fastq.gz\n",
      "Started analysis of 1BS5_S10_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 1BS5_S10_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 1BS5_S10_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 1BS5_S10_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 1BS5_S10_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 1BS5_S10_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 1BS5_S10_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 1BS5_S10_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 1BS5_S10_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 1BS5_S10_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 1BS5_S10_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 1BS5_S10_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 1BS5_S10_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 1BS5_S10_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 1BS5_S10_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 1BS5_S10_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 1BS5_S10_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 1BS5_S10_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 1BS5_S10_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 1BS5_S10_L001_R1_001.fastq.gz\n",
      "Analysis complete for 1BS5_S10_L001_R1_001.fastq.gz\n",
      "Started analysis of 1BS5_S10_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 1BS5_S10_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 1BS5_S10_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 1BS5_S10_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 1BS5_S10_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 1BS5_S10_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 1BS5_S10_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 1BS5_S10_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 1BS5_S10_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 1BS5_S10_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 1BS5_S10_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 1BS5_S10_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 1BS5_S10_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 1BS5_S10_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 1BS5_S10_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 1BS5_S10_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 1BS5_S10_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 1BS5_S10_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 1BS5_S10_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 1BS5_S10_L001_R2_001.fastq.gz\n",
      "Analysis complete for 1BS5_S10_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dDHA1_S59_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dDHA1_S59_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dDHA1_S59_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dDHA1_S59_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dDHA1_S59_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dDHA1_S59_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dDHA1_S59_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dDHA1_S59_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dDHA1_S59_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dDHA1_S59_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dDHA1_S59_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dDHA1_S59_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dDHA1_S59_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dDHA1_S59_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dDHA1_S59_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dDHA1_S59_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dDHA1_S59_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dDHA1_S59_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dDHA1_S59_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dDHA1_S59_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dDHA1_S59_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dDHA1_S59_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dDHA1_S59_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dDHA1_S59_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dDHA1_S59_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dDHA1_S59_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dDHA1_S59_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dDHA1_S59_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dDHA1_S59_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dDHA1_S59_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dDHA1_S59_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dDHA1_S59_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dDHA1_S59_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dDHA1_S59_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dDHA1_S59_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dDHA1_S59_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dDHA1_S59_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dDHA1_S59_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dDHA1_S59_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dDHA1_S59_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dDHA1_S59_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dDHA1_S59_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dDHA2_S60_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dDHA2_S60_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dDHA2_S60_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dDHA2_S60_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dDHA2_S60_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dDHA2_S60_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dDHA2_S60_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dDHA2_S60_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dDHA2_S60_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dDHA2_S60_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dDHA2_S60_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dDHA2_S60_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dDHA2_S60_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dDHA2_S60_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dDHA2_S60_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dDHA2_S60_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dDHA2_S60_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dDHA2_S60_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dDHA2_S60_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dDHA2_S60_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dDHA2_S60_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dDHA2_S60_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dDHA2_S60_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dDHA2_S60_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dDHA2_S60_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dDHA2_S60_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dDHA2_S60_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dDHA2_S60_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dDHA2_S60_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dDHA2_S60_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dDHA2_S60_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dDHA2_S60_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dDHA2_S60_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dDHA2_S60_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dDHA2_S60_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dDHA2_S60_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dDHA2_S60_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dDHA2_S60_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dDHA2_S60_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dDHA2_S60_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dDHA2_S60_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Analysis complete for 5dDHA2_S60_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dDHA3_S61_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dDHA3_S61_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dDHA3_S61_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dDHA3_S61_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dDHA3_S61_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dDHA3_S61_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dDHA3_S61_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dDHA3_S61_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dDHA3_S61_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dDHA3_S61_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dDHA3_S61_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dDHA3_S61_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dDHA3_S61_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dDHA3_S61_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dDHA3_S61_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dDHA3_S61_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dDHA3_S61_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dDHA3_S61_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dDHA3_S61_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dDHA3_S61_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dDHA3_S61_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dDHA3_S61_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dDHA3_S61_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dDHA3_S61_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dDHA3_S61_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dDHA3_S61_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dDHA3_S61_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dDHA3_S61_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dDHA3_S61_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dDHA3_S61_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dDHA3_S61_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dDHA3_S61_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dDHA3_S61_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dDHA3_S61_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dDHA3_S61_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dDHA3_S61_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dDHA3_S61_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dDHA3_S61_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dDHA3_S61_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dDHA3_S61_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dDHA3_S61_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dDHA3_S61_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dDHA4_S62_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dDHA4_S62_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dDHA4_S62_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dDHA4_S62_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dDHA4_S62_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dDHA4_S62_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dDHA4_S62_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dDHA4_S62_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dDHA4_S62_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dDHA4_S62_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dDHA4_S62_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dDHA4_S62_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dDHA4_S62_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dDHA4_S62_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dDHA4_S62_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dDHA4_S62_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dDHA4_S62_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dDHA4_S62_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dDHA4_S62_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dDHA4_S62_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dDHA4_S62_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dDHA4_S62_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dDHA4_S62_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dDHA4_S62_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dDHA4_S62_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dDHA4_S62_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dDHA4_S62_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dDHA4_S62_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dDHA4_S62_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dDHA4_S62_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dDHA4_S62_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dDHA4_S62_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dDHA4_S62_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dDHA4_S62_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dDHA4_S62_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dDHA4_S62_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dDHA4_S62_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dDHA4_S62_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dDHA4_S62_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dDHA4_S62_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dDHA4_S62_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dDHA4_S62_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dDHA5_S63_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dDHA5_S63_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dDHA5_S63_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dDHA5_S63_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dDHA5_S63_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dDHA5_S63_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dDHA5_S63_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dDHA5_S63_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dDHA5_S63_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dDHA5_S63_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dDHA5_S63_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dDHA5_S63_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dDHA5_S63_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dDHA5_S63_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dDHA5_S63_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dDHA5_S63_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dDHA5_S63_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dDHA5_S63_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dDHA5_S63_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dDHA5_S63_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dDHA5_S63_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dDHA5_S63_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dDHA5_S63_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dDHA5_S63_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dDHA5_S63_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dDHA5_S63_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dDHA5_S63_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dDHA5_S63_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dDHA5_S63_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dDHA5_S63_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dDHA5_S63_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dDHA5_S63_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dDHA5_S63_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dDHA5_S63_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dDHA5_S63_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dDHA5_S63_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dDHA5_S63_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dDHA5_S63_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dDHA5_S63_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dDHA5_S63_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dDHA5_S63_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dDHA5_S63_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dDHA6_S64_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dDHA6_S64_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dDHA6_S64_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dDHA6_S64_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dDHA6_S64_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dDHA6_S64_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dDHA6_S64_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dDHA6_S64_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dDHA6_S64_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dDHA6_S64_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dDHA6_S64_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dDHA6_S64_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dDHA6_S64_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dDHA6_S64_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dDHA6_S64_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dDHA6_S64_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dDHA6_S64_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dDHA6_S64_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dDHA6_S64_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dDHA6_S64_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dDHA6_S64_L001_R1_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Started analysis of 5dDHA6_S64_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dDHA6_S64_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dDHA6_S64_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dDHA6_S64_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dDHA6_S64_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dDHA6_S64_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dDHA6_S64_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dDHA6_S64_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dDHA6_S64_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dDHA6_S64_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dDHA6_S64_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dDHA6_S64_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dDHA6_S64_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dDHA6_S64_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dDHA6_S64_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dDHA6_S64_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dDHA6_S64_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dDHA6_S64_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dDHA6_S64_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dDHA6_S64_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dDHA6_S64_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dDHA7_S65_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dDHA7_S65_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dDHA7_S65_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dDHA7_S65_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dDHA7_S65_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dDHA7_S65_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dDHA7_S65_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dDHA7_S65_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dDHA7_S65_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dDHA7_S65_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dDHA7_S65_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dDHA7_S65_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dDHA7_S65_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dDHA7_S65_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dDHA7_S65_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dDHA7_S65_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dDHA7_S65_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dDHA7_S65_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dDHA7_S65_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dDHA7_S65_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dDHA7_S65_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dDHA7_S65_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dDHA7_S65_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dDHA7_S65_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dDHA7_S65_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dDHA7_S65_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dDHA7_S65_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dDHA7_S65_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dDHA7_S65_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dDHA7_S65_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dDHA7_S65_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dDHA7_S65_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dDHA7_S65_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dDHA7_S65_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dDHA7_S65_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dDHA7_S65_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dDHA7_S65_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dDHA7_S65_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dDHA7_S65_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dDHA7_S65_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dDHA7_S65_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dDHA7_S65_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dDHA8_S66_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dDHA8_S66_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dDHA8_S66_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dDHA8_S66_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dDHA8_S66_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dDHA8_S66_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dDHA8_S66_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dDHA8_S66_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dDHA8_S66_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dDHA8_S66_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dDHA8_S66_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dDHA8_S66_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dDHA8_S66_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dDHA8_S66_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dDHA8_S66_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dDHA8_S66_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dDHA8_S66_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dDHA8_S66_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dDHA8_S66_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dDHA8_S66_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dDHA8_S66_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dDHA8_S66_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dDHA8_S66_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dDHA8_S66_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dDHA8_S66_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dDHA8_S66_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dDHA8_S66_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dDHA8_S66_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dDHA8_S66_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dDHA8_S66_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dDHA8_S66_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dDHA8_S66_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dDHA8_S66_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dDHA8_S66_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dDHA8_S66_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dDHA8_S66_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dDHA8_S66_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dDHA8_S66_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dDHA8_S66_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dDHA8_S66_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dDHA8_S66_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dDHA8_S66_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dDHA9_S67_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dDHA9_S67_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dDHA9_S67_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dDHA9_S67_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dDHA9_S67_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dDHA9_S67_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dDHA9_S67_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dDHA9_S67_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dDHA9_S67_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dDHA9_S67_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dDHA9_S67_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dDHA9_S67_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dDHA9_S67_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dDHA9_S67_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dDHA9_S67_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dDHA9_S67_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dDHA9_S67_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dDHA9_S67_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dDHA9_S67_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dDHA9_S67_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dDHA9_S67_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dDHA9_S67_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dDHA9_S67_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dDHA9_S67_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dDHA9_S67_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dDHA9_S67_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dDHA9_S67_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dDHA9_S67_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dDHA9_S67_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dDHA9_S67_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dDHA9_S67_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dDHA9_S67_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dDHA9_S67_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dDHA9_S67_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dDHA9_S67_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dDHA9_S67_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dDHA9_S67_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dDHA9_S67_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dDHA9_S67_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dDHA9_S67_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dDHA9_S67_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dDHA9_S67_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dH1_S28_L001_R1_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 5% complete for 5dH1_S28_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dH1_S28_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dH1_S28_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dH1_S28_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dH1_S28_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dH1_S28_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dH1_S28_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dH1_S28_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dH1_S28_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dH1_S28_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dH1_S28_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dH1_S28_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dH1_S28_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dH1_S28_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dH1_S28_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dH1_S28_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dH1_S28_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dH1_S28_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dH1_S28_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dH1_S28_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dH1_S28_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dH1_S28_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dH1_S28_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dH1_S28_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dH1_S28_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dH1_S28_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dH1_S28_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dH1_S28_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dH1_S28_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dH1_S28_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dH1_S28_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dH1_S28_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dH1_S28_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dH1_S28_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dH1_S28_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dH1_S28_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dH1_S28_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dH1_S28_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dH1_S28_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dH1_S28_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dH1_S28_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dH2_S29_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dH2_S29_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dH2_S29_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dH2_S29_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dH2_S29_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dH2_S29_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dH2_S29_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dH2_S29_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dH2_S29_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dH2_S29_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dH2_S29_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dH2_S29_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dH2_S29_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dH2_S29_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dH2_S29_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dH2_S29_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dH2_S29_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dH2_S29_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dH2_S29_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dH2_S29_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dH2_S29_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dH2_S29_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dH2_S29_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dH2_S29_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dH2_S29_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dH2_S29_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dH2_S29_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dH2_S29_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dH2_S29_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dH2_S29_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dH2_S29_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dH2_S29_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dH2_S29_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dH2_S29_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dH2_S29_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dH2_S29_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dH2_S29_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dH2_S29_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dH2_S29_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dH2_S29_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dH2_S29_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dH2_S29_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dH3_S30_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dH3_S30_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dH3_S30_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dH3_S30_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dH3_S30_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dH3_S30_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dH3_S30_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dH3_S30_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dH3_S30_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dH3_S30_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dH3_S30_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dH3_S30_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dH3_S30_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dH3_S30_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dH3_S30_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dH3_S30_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dH3_S30_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dH3_S30_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dH3_S30_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dH3_S30_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dH3_S30_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dH3_S30_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dH3_S30_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dH3_S30_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dH3_S30_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dH3_S30_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dH3_S30_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dH3_S30_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dH3_S30_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dH3_S30_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dH3_S30_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dH3_S30_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dH3_S30_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dH3_S30_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dH3_S30_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dH3_S30_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dH3_S30_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dH3_S30_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dH3_S30_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dH3_S30_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dH3_S30_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dH3_S30_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dH4_S26_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dH4_S26_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dH4_S26_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dH4_S26_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dH4_S26_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dH4_S26_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dH4_S26_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dH4_S26_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dH4_S26_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dH4_S26_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dH4_S26_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dH4_S26_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dH4_S26_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dH4_S26_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dH4_S26_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dH4_S26_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dH4_S26_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dH4_S26_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dH4_S26_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dH4_S26_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dH4_S26_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dH4_S26_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dH4_S26_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dH4_S26_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dH4_S26_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dH4_S26_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dH4_S26_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dH4_S26_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 35% complete for 5dH4_S26_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dH4_S26_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dH4_S26_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dH4_S26_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dH4_S26_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dH4_S26_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dH4_S26_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dH4_S26_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dH4_S26_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dH4_S26_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dH4_S26_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dH4_S26_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dH4_S26_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dH4_S26_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dH5_S27_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dH5_S27_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dH5_S27_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dH5_S27_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dH5_S27_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dH5_S27_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dH5_S27_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dH5_S27_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dH5_S27_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dH5_S27_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dH5_S27_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dH5_S27_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dH5_S27_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dH5_S27_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dH5_S27_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dH5_S27_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dH5_S27_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dH5_S27_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dH5_S27_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dH5_S27_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dH5_S27_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dH5_S27_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dH5_S27_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dH5_S27_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dH5_S27_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dH5_S27_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dH5_S27_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dH5_S27_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dH5_S27_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dH5_S27_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dH5_S27_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dH5_S27_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dH5_S27_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dH5_S27_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dH5_S27_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dH5_S27_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dH5_S27_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dH5_S27_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dH5_S27_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dH5_S27_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dH5_S27_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dH5_S27_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dIS1_S68_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dIS1_S68_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dIS1_S68_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dIS1_S68_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dIS1_S68_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dIS1_S68_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dIS1_S68_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dIS1_S68_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dIS1_S68_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dIS1_S68_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dIS1_S68_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dIS1_S68_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dIS1_S68_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dIS1_S68_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dIS1_S68_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dIS1_S68_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dIS1_S68_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dIS1_S68_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dIS1_S68_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dIS1_S68_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dIS1_S68_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dIS1_S68_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dIS1_S68_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dIS1_S68_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dIS1_S68_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dIS1_S68_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dIS1_S68_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dIS1_S68_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dIS1_S68_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dIS1_S68_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dIS1_S68_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dIS1_S68_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dIS1_S68_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dIS1_S68_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dIS1_S68_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dIS1_S68_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dIS1_S68_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dIS1_S68_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dIS1_S68_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dIS1_S68_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dIS1_S68_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dIS1_S68_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dIS2_S69_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dIS2_S69_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dIS2_S69_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dIS2_S69_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dIS2_S69_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dIS2_S69_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dIS2_S69_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dIS2_S69_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dIS2_S69_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dIS2_S69_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dIS2_S69_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dIS2_S69_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dIS2_S69_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dIS2_S69_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dIS2_S69_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dIS2_S69_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dIS2_S69_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dIS2_S69_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dIS2_S69_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dIS2_S69_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dIS2_S69_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dIS2_S69_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dIS2_S69_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dIS2_S69_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dIS2_S69_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dIS2_S69_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dIS2_S69_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dIS2_S69_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dIS2_S69_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dIS2_S69_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dIS2_S69_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dIS2_S69_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dIS2_S69_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dIS2_S69_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dIS2_S69_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dIS2_S69_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dIS2_S69_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dIS2_S69_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dIS2_S69_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dIS2_S69_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dIS2_S69_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dIS2_S69_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dIS3_S70_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dIS3_S70_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dIS3_S70_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dIS3_S70_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dIS3_S70_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dIS3_S70_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dIS3_S70_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dIS3_S70_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dIS3_S70_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dIS3_S70_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dIS3_S70_L001_R1_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 55% complete for 5dIS3_S70_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dIS3_S70_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dIS3_S70_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dIS3_S70_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dIS3_S70_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dIS3_S70_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dIS3_S70_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dIS3_S70_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dIS3_S70_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dIS3_S70_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dIS3_S70_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dIS3_S70_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dIS3_S70_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dIS3_S70_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dIS3_S70_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dIS3_S70_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dIS3_S70_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dIS3_S70_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dIS3_S70_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dIS3_S70_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dIS3_S70_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dIS3_S70_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dIS3_S70_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dIS3_S70_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dIS3_S70_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dIS3_S70_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dIS3_S70_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dIS3_S70_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dIS3_S70_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dIS3_S70_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dIS3_S70_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dIS4_S71_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dIS4_S71_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dIS4_S71_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dIS4_S71_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dIS4_S71_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dIS4_S71_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dIS4_S71_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dIS4_S71_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dIS4_S71_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dIS4_S71_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dIS4_S71_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dIS4_S71_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dIS4_S71_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dIS4_S71_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dIS4_S71_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dIS4_S71_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dIS4_S71_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dIS4_S71_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dIS4_S71_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dIS4_S71_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dIS4_S71_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dIS4_S71_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dIS4_S71_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dIS4_S71_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dIS4_S71_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dIS4_S71_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dIS4_S71_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dIS4_S71_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dIS4_S71_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dIS4_S71_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dIS4_S71_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dIS4_S71_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dIS4_S71_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dIS4_S71_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dIS4_S71_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dIS4_S71_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dIS4_S71_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dIS4_S71_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dIS4_S71_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dIS4_S71_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dIS4_S71_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dIS4_S71_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dIS6_S72_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dIS6_S72_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dIS6_S72_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dIS6_S72_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dIS6_S72_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dIS6_S72_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dIS6_S72_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dIS6_S72_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dIS6_S72_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dIS6_S72_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dIS6_S72_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dIS6_S72_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dIS6_S72_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dIS6_S72_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dIS6_S72_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dIS6_S72_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dIS6_S72_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dIS6_S72_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dIS6_S72_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dIS6_S72_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dIS6_S72_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dIS6_S72_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dIS6_S72_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dIS6_S72_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dIS6_S72_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dIS6_S72_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dIS6_S72_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dIS6_S72_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dIS6_S72_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dIS6_S72_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dIS6_S72_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dIS6_S72_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dIS6_S72_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dIS6_S72_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dIS6_S72_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dIS6_S72_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dIS6_S72_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dIS6_S72_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dIS6_S72_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dIS6_S72_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dIS6_S72_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dIS6_S72_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dIS7_S73_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dIS7_S73_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dIS7_S73_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dIS7_S73_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dIS7_S73_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dIS7_S73_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dIS7_S73_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dIS7_S73_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dIS7_S73_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dIS7_S73_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dIS7_S73_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dIS7_S73_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dIS7_S73_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dIS7_S73_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dIS7_S73_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dIS7_S73_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dIS7_S73_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dIS7_S73_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dIS7_S73_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dIS7_S73_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dIS7_S73_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dIS7_S73_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dIS7_S73_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dIS7_S73_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dIS7_S73_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dIS7_S73_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dIS7_S73_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dIS7_S73_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dIS7_S73_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dIS7_S73_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dIS7_S73_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dIS7_S73_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dIS7_S73_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dIS7_S73_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dIS7_S73_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 70% complete for 5dIS7_S73_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dIS7_S73_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dIS7_S73_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dIS7_S73_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dIS7_S73_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dIS7_S73_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dIS7_S73_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dIS8_S74_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dIS8_S74_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dIS8_S74_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dIS8_S74_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dIS8_S74_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dIS8_S74_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dIS8_S74_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dIS8_S74_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dIS8_S74_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dIS8_S74_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dIS8_S74_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dIS8_S74_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dIS8_S74_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dIS8_S74_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dIS8_S74_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dIS8_S74_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dIS8_S74_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dIS8_S74_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dIS8_S74_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dIS8_S74_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dIS8_S74_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dIS8_S74_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dIS8_S74_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dIS8_S74_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dIS8_S74_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dIS8_S74_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dIS8_S74_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dIS8_S74_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dIS8_S74_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dIS8_S74_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dIS8_S74_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dIS8_S74_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dIS8_S74_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dIS8_S74_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dIS8_S74_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dIS8_S74_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dIS8_S74_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dIS8_S74_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dIS8_S74_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dIS8_S74_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dIS8_S74_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dIS8_S74_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dIS9_S75_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dIS9_S75_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dIS9_S75_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dIS9_S75_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dIS9_S75_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dIS9_S75_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dIS9_S75_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dIS9_S75_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dIS9_S75_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dIS9_S75_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dIS9_S75_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dIS9_S75_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dIS9_S75_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dIS9_S75_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dIS9_S75_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dIS9_S75_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dIS9_S75_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dIS9_S75_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dIS9_S75_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dIS9_S75_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dIS9_S75_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dIS9_S75_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dIS9_S75_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dIS9_S75_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dIS9_S75_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dIS9_S75_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dIS9_S75_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dIS9_S75_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dIS9_S75_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dIS9_S75_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dIS9_S75_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dIS9_S75_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dIS9_S75_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dIS9_S75_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dIS9_S75_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dIS9_S75_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dIS9_S75_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dIS9_S75_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dIS9_S75_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dIS9_S75_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dIS9_S75_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dIS9_S75_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dMGO1_S51_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dMGO1_S51_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dMGO1_S51_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dMGO1_S51_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dMGO1_S51_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dMGO1_S51_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dMGO1_S51_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dMGO1_S51_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dMGO1_S51_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dMGO1_S51_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dMGO1_S51_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dMGO1_S51_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dMGO1_S51_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dMGO1_S51_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dMGO1_S51_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dMGO1_S51_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dMGO1_S51_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dMGO1_S51_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dMGO1_S51_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dMGO1_S51_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dMGO1_S51_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dMGO1_S51_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dMGO1_S51_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dMGO1_S51_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dMGO1_S51_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dMGO1_S51_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dMGO1_S51_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dMGO1_S51_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dMGO1_S51_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dMGO1_S51_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dMGO1_S51_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dMGO1_S51_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dMGO1_S51_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dMGO1_S51_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dMGO1_S51_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dMGO1_S51_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dMGO1_S51_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dMGO1_S51_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dMGO1_S51_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dMGO1_S51_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dMGO1_S51_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dMGO1_S51_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dMGO2_S52_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dMGO2_S52_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dMGO2_S52_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dMGO2_S52_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dMGO2_S52_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dMGO2_S52_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dMGO2_S52_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dMGO2_S52_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dMGO2_S52_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dMGO2_S52_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dMGO2_S52_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dMGO2_S52_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dMGO2_S52_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dMGO2_S52_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dMGO2_S52_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dMGO2_S52_L001_R1_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 80% complete for 5dMGO2_S52_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dMGO2_S52_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dMGO2_S52_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dMGO2_S52_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dMGO2_S52_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dMGO2_S52_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dMGO2_S52_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dMGO2_S52_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dMGO2_S52_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dMGO2_S52_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dMGO2_S52_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dMGO2_S52_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dMGO2_S52_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dMGO2_S52_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dMGO2_S52_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dMGO2_S52_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dMGO2_S52_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dMGO2_S52_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dMGO2_S52_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dMGO2_S52_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dMGO2_S52_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dMGO2_S52_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dMGO2_S52_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dMGO2_S52_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dMGO2_S52_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dMGO2_S52_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dMGO3_S53_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dMGO3_S53_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dMGO3_S53_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dMGO3_S53_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dMGO3_S53_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dMGO3_S53_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dMGO3_S53_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dMGO3_S53_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dMGO3_S53_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dMGO3_S53_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dMGO3_S53_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dMGO3_S53_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dMGO3_S53_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dMGO3_S53_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dMGO3_S53_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dMGO3_S53_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dMGO3_S53_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dMGO3_S53_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dMGO3_S53_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dMGO3_S53_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dMGO3_S53_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dMGO3_S53_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dMGO3_S53_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dMGO3_S53_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dMGO3_S53_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dMGO3_S53_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dMGO3_S53_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dMGO3_S53_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dMGO3_S53_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dMGO3_S53_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dMGO3_S53_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dMGO3_S53_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dMGO3_S53_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dMGO3_S53_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dMGO3_S53_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dMGO3_S53_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dMGO3_S53_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dMGO3_S53_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dMGO3_S53_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dMGO3_S53_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dMGO3_S53_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dMGO3_S53_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dMGO4_S54_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dMGO4_S54_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dMGO4_S54_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dMGO4_S54_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dMGO4_S54_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dMGO4_S54_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dMGO4_S54_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dMGO4_S54_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dMGO4_S54_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dMGO4_S54_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dMGO4_S54_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dMGO4_S54_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dMGO4_S54_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dMGO4_S54_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dMGO4_S54_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dMGO4_S54_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dMGO4_S54_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dMGO4_S54_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dMGO4_S54_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dMGO4_S54_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dMGO4_S54_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dMGO4_S54_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dMGO4_S54_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dMGO4_S54_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dMGO4_S54_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dMGO4_S54_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dMGO4_S54_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dMGO4_S54_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dMGO4_S54_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dMGO4_S54_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dMGO4_S54_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dMGO4_S54_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dMGO4_S54_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dMGO4_S54_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dMGO4_S54_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dMGO4_S54_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dMGO4_S54_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dMGO4_S54_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dMGO4_S54_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dMGO4_S54_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dMGO4_S54_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dMGO4_S54_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dMGO6_S55_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dMGO6_S55_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dMGO6_S55_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dMGO6_S55_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dMGO6_S55_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dMGO6_S55_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dMGO6_S55_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dMGO6_S55_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dMGO6_S55_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dMGO6_S55_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dMGO6_S55_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dMGO6_S55_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dMGO6_S55_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dMGO6_S55_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dMGO6_S55_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dMGO6_S55_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dMGO6_S55_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dMGO6_S55_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dMGO6_S55_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dMGO6_S55_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dMGO6_S55_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dMGO6_S55_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dMGO6_S55_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dMGO6_S55_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dMGO6_S55_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dMGO6_S55_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dMGO6_S55_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dMGO6_S55_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dMGO6_S55_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dMGO6_S55_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dMGO6_S55_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dMGO6_S55_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dMGO6_S55_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dMGO6_S55_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dMGO6_S55_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dMGO6_S55_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dMGO6_S55_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dMGO6_S55_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 85% complete for 5dMGO6_S55_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dMGO6_S55_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dMGO6_S55_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dMGO6_S55_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dMGO7_S56_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dMGO7_S56_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dMGO7_S56_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dMGO7_S56_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dMGO7_S56_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dMGO7_S56_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dMGO7_S56_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dMGO7_S56_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dMGO7_S56_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dMGO7_S56_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dMGO7_S56_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dMGO7_S56_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dMGO7_S56_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dMGO7_S56_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dMGO7_S56_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dMGO7_S56_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dMGO7_S56_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dMGO7_S56_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dMGO7_S56_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dMGO7_S56_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dMGO7_S56_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dMGO7_S56_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dMGO7_S56_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dMGO7_S56_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dMGO7_S56_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dMGO7_S56_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dMGO7_S56_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dMGO7_S56_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dMGO7_S56_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dMGO7_S56_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dMGO7_S56_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dMGO7_S56_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dMGO7_S56_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dMGO7_S56_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dMGO7_S56_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dMGO7_S56_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dMGO7_S56_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dMGO7_S56_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dMGO7_S56_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dMGO7_S56_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dMGO7_S56_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dMGO7_S56_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dMGO8_S57_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dMGO8_S57_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dMGO8_S57_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dMGO8_S57_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dMGO8_S57_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dMGO8_S57_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dMGO8_S57_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dMGO8_S57_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dMGO8_S57_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dMGO8_S57_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dMGO8_S57_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dMGO8_S57_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dMGO8_S57_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dMGO8_S57_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dMGO8_S57_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dMGO8_S57_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dMGO8_S57_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dMGO8_S57_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dMGO8_S57_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dMGO8_S57_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dMGO8_S57_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dMGO8_S57_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dMGO8_S57_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dMGO8_S57_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dMGO8_S57_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dMGO8_S57_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dMGO8_S57_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dMGO8_S57_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dMGO8_S57_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dMGO8_S57_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dMGO8_S57_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dMGO8_S57_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dMGO8_S57_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dMGO8_S57_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dMGO8_S57_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dMGO8_S57_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dMGO8_S57_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dMGO8_S57_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dMGO8_S57_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dMGO8_S57_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dMGO8_S57_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dMGO8_S57_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dMGO9_S58_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dMGO9_S58_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dMGO9_S58_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dMGO9_S58_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dMGO9_S58_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dMGO9_S58_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dMGO9_S58_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dMGO9_S58_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dMGO9_S58_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dMGO9_S58_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dMGO9_S58_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dMGO9_S58_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dMGO9_S58_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dMGO9_S58_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dMGO9_S58_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dMGO9_S58_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dMGO9_S58_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dMGO9_S58_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dMGO9_S58_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dMGO9_S58_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dMGO9_S58_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dMGO9_S58_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dMGO9_S58_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dMGO9_S58_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dMGO9_S58_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dMGO9_S58_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dMGO9_S58_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dMGO9_S58_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dMGO9_S58_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dMGO9_S58_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dMGO9_S58_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dMGO9_S58_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dMGO9_S58_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dMGO9_S58_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dMGO9_S58_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dMGO9_S58_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dMGO9_S58_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dMGO9_S58_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dMGO9_S58_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dMGO9_S58_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dMGO9_S58_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dMGO9_S58_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dMH15-1_S35_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dMH15-1_S35_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dMH15-1_S35_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dMH15-1_S35_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dMH15-1_S35_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dMH15-1_S35_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dMH15-1_S35_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dMH15-1_S35_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dMH15-1_S35_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dMH15-1_S35_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dMH15-1_S35_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dMH15-1_S35_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dMH15-1_S35_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dMH15-1_S35_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dMH15-1_S35_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dMH15-1_S35_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dMH15-1_S35_L001_R1_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 85% complete for 5dMH15-1_S35_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dMH15-1_S35_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dMH15-1_S35_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dMH15-1_S35_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dMH15-1_S35_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dMH15-1_S35_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dMH15-1_S35_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dMH15-1_S35_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dMH15-1_S35_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dMH15-1_S35_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dMH15-1_S35_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dMH15-1_S35_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dMH15-1_S35_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dMH15-1_S35_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dMH15-1_S35_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dMH15-1_S35_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dMH15-1_S35_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dMH15-1_S35_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dMH15-1_S35_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dMH15-1_S35_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dMH15-1_S35_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dMH15-1_S35_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dMH15-1_S35_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dMH15-1_S35_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dMH15-1_S35_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dMH15-2_S36_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dMH15-2_S36_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dMH15-2_S36_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dMH15-2_S36_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dMH15-2_S36_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dMH15-2_S36_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dMH15-2_S36_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dMH15-2_S36_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dMH15-2_S36_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dMH15-2_S36_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dMH15-2_S36_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dMH15-2_S36_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dMH15-2_S36_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dMH15-2_S36_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dMH15-2_S36_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dMH15-2_S36_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dMH15-2_S36_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dMH15-2_S36_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dMH15-2_S36_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dMH15-2_S36_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dMH15-2_S36_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dMH15-2_S36_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dMH15-2_S36_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dMH15-2_S36_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dMH15-2_S36_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dMH15-2_S36_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dMH15-2_S36_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dMH15-2_S36_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dMH15-2_S36_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dMH15-2_S36_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dMH15-2_S36_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dMH15-2_S36_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dMH15-2_S36_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dMH15-2_S36_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dMH15-2_S36_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dMH15-2_S36_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dMH15-2_S36_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dMH15-2_S36_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dMH15-2_S36_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dMH15-2_S36_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dMH15-2_S36_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dMH15-2_S36_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dMH15-3_S37_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dMH15-3_S37_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dMH15-3_S37_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dMH15-3_S37_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dMH15-3_S37_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dMH15-3_S37_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dMH15-3_S37_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dMH15-3_S37_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dMH15-3_S37_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dMH15-3_S37_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dMH15-3_S37_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dMH15-3_S37_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dMH15-3_S37_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dMH15-3_S37_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dMH15-3_S37_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dMH15-3_S37_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dMH15-3_S37_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dMH15-3_S37_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dMH15-3_S37_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dMH15-3_S37_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dMH15-3_S37_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dMH15-3_S37_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dMH15-3_S37_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dMH15-3_S37_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dMH15-3_S37_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dMH15-3_S37_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dMH15-3_S37_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dMH15-3_S37_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dMH15-3_S37_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dMH15-3_S37_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dMH15-3_S37_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dMH15-3_S37_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dMH15-3_S37_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dMH15-3_S37_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dMH15-3_S37_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dMH15-3_S37_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dMH15-3_S37_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dMH15-3_S37_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dMH15-3_S37_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dMH15-3_S37_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dMH15-3_S37_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dMH15-3_S37_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dMH15-4_S38_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dMH15-4_S38_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dMH15-4_S38_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dMH15-4_S38_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dMH15-4_S38_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dMH15-4_S38_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dMH15-4_S38_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dMH15-4_S38_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dMH15-4_S38_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dMH15-4_S38_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dMH15-4_S38_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dMH15-4_S38_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dMH15-4_S38_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dMH15-4_S38_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dMH15-4_S38_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dMH15-4_S38_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dMH15-4_S38_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dMH15-4_S38_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dMH15-4_S38_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dMH15-4_S38_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dMH15-4_S38_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dMH15-4_S38_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dMH15-4_S38_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dMH15-4_S38_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dMH15-4_S38_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dMH15-4_S38_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dMH15-4_S38_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dMH15-4_S38_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dMH15-4_S38_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dMH15-4_S38_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dMH15-4_S38_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dMH15-4_S38_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dMH15-4_S38_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dMH15-4_S38_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 65% complete for 5dMH15-4_S38_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dMH15-4_S38_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dMH15-4_S38_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dMH15-4_S38_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dMH15-4_S38_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dMH15-4_S38_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dMH15-4_S38_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dMH15-4_S38_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dMH15-6_S39_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dMH15-6_S39_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dMH15-6_S39_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dMH15-6_S39_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dMH15-6_S39_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dMH15-6_S39_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dMH15-6_S39_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dMH15-6_S39_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dMH15-6_S39_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dMH15-6_S39_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dMH15-6_S39_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dMH15-6_S39_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dMH15-6_S39_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dMH15-6_S39_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dMH15-6_S39_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dMH15-6_S39_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dMH15-6_S39_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dMH15-6_S39_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dMH15-6_S39_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dMH15-6_S39_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dMH15-6_S39_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dMH15-6_S39_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dMH15-6_S39_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dMH15-6_S39_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dMH15-6_S39_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dMH15-6_S39_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dMH15-6_S39_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dMH15-6_S39_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dMH15-6_S39_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dMH15-6_S39_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dMH15-6_S39_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dMH15-6_S39_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dMH15-6_S39_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dMH15-6_S39_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dMH15-6_S39_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dMH15-6_S39_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dMH15-6_S39_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dMH15-6_S39_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dMH15-6_S39_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dMH15-6_S39_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dMH15-6_S39_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dMH15-6_S39_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dMH15-7_S40_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dMH15-7_S40_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dMH15-7_S40_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dMH15-7_S40_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dMH15-7_S40_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dMH15-7_S40_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dMH15-7_S40_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dMH15-7_S40_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dMH15-7_S40_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dMH15-7_S40_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dMH15-7_S40_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dMH15-7_S40_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dMH15-7_S40_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dMH15-7_S40_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dMH15-7_S40_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dMH15-7_S40_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dMH15-7_S40_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dMH15-7_S40_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dMH15-7_S40_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dMH15-7_S40_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dMH15-7_S40_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dMH15-7_S40_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dMH15-7_S40_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dMH15-7_S40_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dMH15-7_S40_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dMH15-7_S40_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dMH15-7_S40_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dMH15-7_S40_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dMH15-7_S40_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dMH15-7_S40_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dMH15-7_S40_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dMH15-7_S40_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dMH15-7_S40_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dMH15-7_S40_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dMH15-7_S40_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dMH15-7_S40_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dMH15-7_S40_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dMH15-7_S40_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dMH15-7_S40_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dMH15-7_S40_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dMH15-7_S40_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dMH15-7_S40_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dMH15-8_S41_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dMH15-8_S41_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dMH15-8_S41_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dMH15-8_S41_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dMH15-8_S41_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dMH15-8_S41_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dMH15-8_S41_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dMH15-8_S41_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dMH15-8_S41_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dMH15-8_S41_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dMH15-8_S41_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dMH15-8_S41_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dMH15-8_S41_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dMH15-8_S41_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dMH15-8_S41_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dMH15-8_S41_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dMH15-8_S41_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dMH15-8_S41_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dMH15-8_S41_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dMH15-8_S41_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dMH15-8_S41_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dMH15-8_S41_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dMH15-8_S41_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dMH15-8_S41_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dMH15-8_S41_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dMH15-8_S41_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dMH15-8_S41_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dMH15-8_S41_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dMH15-8_S41_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dMH15-8_S41_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dMH15-8_S41_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dMH15-8_S41_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dMH15-8_S41_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dMH15-8_S41_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dMH15-8_S41_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dMH15-8_S41_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dMH15-8_S41_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dMH15-8_S41_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dMH15-8_S41_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dMH15-8_S41_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dMH15-8_S41_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dMH15-8_S41_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dMH15-9_S42_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dMH15-9_S42_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dMH15-9_S42_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dMH15-9_S42_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dMH15-9_S42_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dMH15-9_S42_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dMH15-9_S42_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dMH15-9_S42_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dMH15-9_S42_L001_R1_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 45% complete for 5dMH15-9_S42_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dMH15-9_S42_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dMH15-9_S42_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dMH15-9_S42_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dMH15-9_S42_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dMH15-9_S42_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dMH15-9_S42_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dMH15-9_S42_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dMH15-9_S42_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dMH15-9_S42_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dMH15-9_S42_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dMH15-9_S42_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dMH15-9_S42_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dMH15-9_S42_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dMH15-9_S42_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dMH15-9_S42_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dMH15-9_S42_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dMH15-9_S42_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dMH15-9_S42_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dMH15-9_S42_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dMH15-9_S42_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dMH15-9_S42_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dMH15-9_S42_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dMH15-9_S42_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dMH15-9_S42_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dMH15-9_S42_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dMH15-9_S42_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dMH15-9_S42_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dMH15-9_S42_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dMH15-9_S42_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dMH15-9_S42_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dMH15-9_S42_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dMH15-9_S42_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dMH17-1_S43_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dMH17-1_S43_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dMH17-1_S43_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dMH17-1_S43_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dMH17-1_S43_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dMH17-1_S43_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dMH17-1_S43_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dMH17-1_S43_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dMH17-1_S43_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dMH17-1_S43_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dMH17-1_S43_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dMH17-1_S43_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dMH17-1_S43_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dMH17-1_S43_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dMH17-1_S43_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dMH17-1_S43_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dMH17-1_S43_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dMH17-1_S43_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dMH17-1_S43_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dMH17-1_S43_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dMH17-1_S43_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dMH17-1_S43_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dMH17-1_S43_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dMH17-1_S43_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dMH17-1_S43_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dMH17-1_S43_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dMH17-1_S43_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dMH17-1_S43_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dMH17-1_S43_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dMH17-1_S43_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dMH17-1_S43_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dMH17-1_S43_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dMH17-1_S43_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dMH17-1_S43_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dMH17-1_S43_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dMH17-1_S43_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dMH17-1_S43_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dMH17-1_S43_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dMH17-1_S43_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dMH17-1_S43_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dMH17-1_S43_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dMH17-1_S43_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dMH17-2_S44_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dMH17-2_S44_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dMH17-2_S44_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dMH17-2_S44_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dMH17-2_S44_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dMH17-2_S44_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dMH17-2_S44_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dMH17-2_S44_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dMH17-2_S44_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dMH17-2_S44_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dMH17-2_S44_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dMH17-2_S44_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dMH17-2_S44_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dMH17-2_S44_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dMH17-2_S44_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dMH17-2_S44_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dMH17-2_S44_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dMH17-2_S44_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dMH17-2_S44_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dMH17-2_S44_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dMH17-2_S44_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dMH17-2_S44_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dMH17-2_S44_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dMH17-2_S44_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dMH17-2_S44_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dMH17-2_S44_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dMH17-2_S44_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dMH17-2_S44_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dMH17-2_S44_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dMH17-2_S44_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dMH17-2_S44_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dMH17-2_S44_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dMH17-2_S44_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dMH17-2_S44_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dMH17-2_S44_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dMH17-2_S44_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dMH17-2_S44_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dMH17-2_S44_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dMH17-2_S44_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dMH17-2_S44_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dMH17-2_S44_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dMH17-2_S44_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dMH17-3_S45_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dMH17-3_S45_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dMH17-3_S45_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dMH17-3_S45_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dMH17-3_S45_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dMH17-3_S45_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dMH17-3_S45_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dMH17-3_S45_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dMH17-3_S45_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dMH17-3_S45_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dMH17-3_S45_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dMH17-3_S45_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dMH17-3_S45_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dMH17-3_S45_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dMH17-3_S45_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dMH17-3_S45_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dMH17-3_S45_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dMH17-3_S45_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dMH17-3_S45_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dMH17-3_S45_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dMH17-3_S45_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dMH17-3_S45_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dMH17-3_S45_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dMH17-3_S45_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dMH17-3_S45_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dMH17-3_S45_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 25% complete for 5dMH17-3_S45_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dMH17-3_S45_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dMH17-3_S45_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dMH17-3_S45_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dMH17-3_S45_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dMH17-3_S45_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dMH17-3_S45_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dMH17-3_S45_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dMH17-3_S45_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dMH17-3_S45_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dMH17-3_S45_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dMH17-3_S45_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dMH17-3_S45_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dMH17-3_S45_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dMH17-3_S45_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dMH17-3_S45_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dMH17-4_S46_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dMH17-4_S46_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dMH17-4_S46_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dMH17-4_S46_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dMH17-4_S46_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dMH17-4_S46_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dMH17-4_S46_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dMH17-4_S46_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dMH17-4_S46_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dMH17-4_S46_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dMH17-4_S46_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dMH17-4_S46_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dMH17-4_S46_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dMH17-4_S46_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dMH17-4_S46_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dMH17-4_S46_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dMH17-4_S46_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dMH17-4_S46_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dMH17-4_S46_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dMH17-4_S46_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dMH17-4_S46_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dMH17-4_S46_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dMH17-4_S46_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dMH17-4_S46_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dMH17-4_S46_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dMH17-4_S46_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dMH17-4_S46_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dMH17-4_S46_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dMH17-4_S46_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dMH17-4_S46_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dMH17-4_S46_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dMH17-4_S46_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dMH17-4_S46_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dMH17-4_S46_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dMH17-4_S46_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dMH17-4_S46_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dMH17-4_S46_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dMH17-4_S46_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dMH17-4_S46_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dMH17-4_S46_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dMH17-4_S46_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dMH17-4_S46_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dMH17-6_S47_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dMH17-6_S47_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dMH17-6_S47_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dMH17-6_S47_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dMH17-6_S47_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dMH17-6_S47_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dMH17-6_S47_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dMH17-6_S47_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dMH17-6_S47_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dMH17-6_S47_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dMH17-6_S47_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dMH17-6_S47_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dMH17-6_S47_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dMH17-6_S47_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dMH17-6_S47_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dMH17-6_S47_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dMH17-6_S47_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dMH17-6_S47_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dMH17-6_S47_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dMH17-6_S47_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dMH17-6_S47_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dMH17-6_S47_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dMH17-6_S47_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dMH17-6_S47_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dMH17-6_S47_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dMH17-6_S47_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dMH17-6_S47_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dMH17-6_S47_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dMH17-6_S47_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dMH17-6_S47_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dMH17-6_S47_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dMH17-6_S47_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dMH17-6_S47_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dMH17-6_S47_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dMH17-6_S47_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dMH17-6_S47_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dMH17-6_S47_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dMH17-6_S47_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dMH17-6_S47_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dMH17-6_S47_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dMH17-6_S47_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dMH17-6_S47_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dMH17-7_S48_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dMH17-7_S48_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dMH17-7_S48_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dMH17-7_S48_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dMH17-7_S48_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dMH17-7_S48_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dMH17-7_S48_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dMH17-7_S48_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dMH17-7_S48_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dMH17-7_S48_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dMH17-7_S48_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dMH17-7_S48_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dMH17-7_S48_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dMH17-7_S48_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dMH17-7_S48_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dMH17-7_S48_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dMH17-7_S48_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dMH17-7_S48_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dMH17-7_S48_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dMH17-7_S48_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dMH17-7_S48_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dMH17-7_S48_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dMH17-7_S48_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dMH17-7_S48_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dMH17-7_S48_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dMH17-7_S48_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dMH17-7_S48_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dMH17-7_S48_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dMH17-7_S48_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dMH17-7_S48_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dMH17-7_S48_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dMH17-7_S48_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dMH17-7_S48_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dMH17-7_S48_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dMH17-7_S48_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dMH17-7_S48_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dMH17-7_S48_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dMH17-7_S48_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dMH17-7_S48_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dMH17-7_S48_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dMH17-7_S48_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dMH17-7_S48_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dMH17-8_S49_L001_R1_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 5% complete for 5dMH17-8_S49_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dMH17-8_S49_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dMH17-8_S49_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dMH17-8_S49_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dMH17-8_S49_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dMH17-8_S49_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dMH17-8_S49_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dMH17-8_S49_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dMH17-8_S49_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dMH17-8_S49_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dMH17-8_S49_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dMH17-8_S49_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dMH17-8_S49_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dMH17-8_S49_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dMH17-8_S49_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dMH17-8_S49_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dMH17-8_S49_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dMH17-8_S49_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dMH17-8_S49_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dMH17-8_S49_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dMH17-8_S49_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dMH17-8_S49_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dMH17-8_S49_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dMH17-8_S49_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dMH17-8_S49_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dMH17-8_S49_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dMH17-8_S49_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dMH17-8_S49_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dMH17-8_S49_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dMH17-8_S49_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dMH17-8_S49_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dMH17-8_S49_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dMH17-8_S49_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dMH17-8_S49_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dMH17-8_S49_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dMH17-8_S49_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dMH17-8_S49_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dMH17-8_S49_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dMH17-8_S49_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dMH17-8_S49_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dMH17-8_S49_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dMH17-9_S50_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dMH17-9_S50_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dMH17-9_S50_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dMH17-9_S50_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dMH17-9_S50_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dMH17-9_S50_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dMH17-9_S50_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dMH17-9_S50_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dMH17-9_S50_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dMH17-9_S50_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dMH17-9_S50_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dMH17-9_S50_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dMH17-9_S50_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dMH17-9_S50_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dMH17-9_S50_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dMH17-9_S50_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dMH17-9_S50_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dMH17-9_S50_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dMH17-9_S50_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dMH17-9_S50_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dMH17-9_S50_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dMH17-9_S50_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dMH17-9_S50_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dMH17-9_S50_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dMH17-9_S50_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dMH17-9_S50_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dMH17-9_S50_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dMH17-9_S50_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dMH17-9_S50_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dMH17-9_S50_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dMH17-9_S50_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dMH17-9_S50_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dMH17-9_S50_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dMH17-9_S50_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dMH17-9_S50_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dMH17-9_S50_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dMH17-9_S50_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dMH17-9_S50_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dMH17-9_S50_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dMH17-9_S50_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dMH17-9_S50_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dMH17-9_S50_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dSI1_S31_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dSI1_S31_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dSI1_S31_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dSI1_S31_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dSI1_S31_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dSI1_S31_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dSI1_S31_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dSI1_S31_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dSI1_S31_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dSI1_S31_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dSI1_S31_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dSI1_S31_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dSI1_S31_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dSI1_S31_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dSI1_S31_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dSI1_S31_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dSI1_S31_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dSI1_S31_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dSI1_S31_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dSI1_S31_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dSI1_S31_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dSI1_S31_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dSI1_S31_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dSI1_S31_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dSI1_S31_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dSI1_S31_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dSI1_S31_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dSI1_S31_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dSI1_S31_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dSI1_S31_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dSI1_S31_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dSI1_S31_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dSI1_S31_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dSI1_S31_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dSI1_S31_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dSI1_S31_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dSI1_S31_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dSI1_S31_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dSI1_S31_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dSI1_S31_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dSI1_S31_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dSI1_S31_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dSI2_S32_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dSI2_S32_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dSI2_S32_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dSI2_S32_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dSI2_S32_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dSI2_S32_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dSI2_S32_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dSI2_S32_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dSI2_S32_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dSI2_S32_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dSI2_S32_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dSI2_S32_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dSI2_S32_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dSI2_S32_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dSI2_S32_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dSI2_S32_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dSI2_S32_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dSI2_S32_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dSI2_S32_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dSI2_S32_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dSI2_S32_L001_R1_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Started analysis of 5dSI2_S32_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dSI2_S32_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dSI2_S32_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dSI2_S32_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dSI2_S32_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dSI2_S32_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dSI2_S32_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dSI2_S32_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dSI2_S32_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dSI2_S32_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dSI2_S32_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dSI2_S32_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dSI2_S32_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dSI2_S32_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dSI2_S32_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dSI2_S32_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dSI2_S32_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dSI2_S32_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dSI2_S32_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dSI2_S32_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dSI2_S32_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dSI3_S33_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dSI3_S33_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dSI3_S33_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dSI3_S33_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dSI3_S33_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dSI3_S33_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dSI3_S33_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dSI3_S33_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dSI3_S33_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dSI3_S33_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dSI3_S33_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dSI3_S33_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dSI3_S33_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dSI3_S33_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dSI3_S33_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dSI3_S33_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dSI3_S33_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dSI3_S33_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dSI3_S33_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dSI3_S33_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dSI3_S33_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dSI3_S33_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dSI3_S33_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dSI3_S33_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dSI3_S33_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dSI3_S33_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dSI3_S33_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dSI3_S33_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dSI3_S33_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dSI3_S33_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dSI3_S33_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dSI3_S33_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dSI3_S33_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dSI3_S33_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dSI3_S33_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dSI3_S33_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dSI3_S33_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dSI3_S33_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dSI3_S33_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dSI3_S33_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dSI3_S33_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dSI3_S33_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dSI4_S34_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dSI4_S34_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dSI4_S34_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dSI4_S34_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dSI4_S34_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dSI4_S34_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dSI4_S34_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dSI4_S34_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dSI4_S34_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dSI4_S34_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dSI4_S34_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dSI4_S34_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dSI4_S34_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dSI4_S34_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dSI4_S34_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dSI4_S34_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dSI4_S34_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dSI4_S34_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dSI4_S34_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dSI4_S34_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dSI4_S34_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dSI4_S34_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dSI4_S34_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dSI4_S34_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dSI4_S34_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dSI4_S34_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dSI4_S34_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dSI4_S34_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dSI4_S34_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dSI4_S34_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dSI4_S34_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dSI4_S34_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dSI4_S34_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dSI4_S34_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dSI4_S34_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dSI4_S34_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dSI4_S34_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dSI4_S34_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dSI4_S34_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dSI4_S34_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dSI4_S34_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dSI4_S34_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dSI5_S21_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dSI5_S21_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dSI5_S21_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dSI5_S21_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dSI5_S21_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dSI5_S21_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dSI5_S21_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dSI5_S21_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dSI5_S21_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dSI5_S21_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dSI5_S21_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dSI5_S21_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dSI5_S21_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dSI5_S21_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dSI5_S21_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dSI5_S21_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dSI5_S21_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dSI5_S21_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dSI5_S21_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dSI5_S21_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dSI5_S21_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dSI5_S21_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dSI5_S21_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dSI5_S21_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dSI5_S21_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dSI5_S21_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dSI5_S21_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dSI5_S21_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dSI5_S21_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dSI5_S21_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dSI5_S21_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dSI5_S21_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dSI5_S21_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dSI5_S21_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dSI5_S21_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dSI5_S21_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dSI5_S21_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dSI5_S21_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dSI5_S21_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dSI5_S21_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dSI5_S21_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dSI5_S21_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dSI6_S22_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dSI6_S22_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dSI6_S22_L001_R1_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 15% complete for 5dSI6_S22_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dSI6_S22_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dSI6_S22_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dSI6_S22_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dSI6_S22_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dSI6_S22_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dSI6_S22_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dSI6_S22_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dSI6_S22_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dSI6_S22_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dSI6_S22_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dSI6_S22_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dSI6_S22_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dSI6_S22_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dSI6_S22_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dSI6_S22_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dSI6_S22_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dSI6_S22_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dSI6_S22_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dSI6_S22_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dSI6_S22_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dSI6_S22_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dSI6_S22_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dSI6_S22_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dSI6_S22_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dSI6_S22_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dSI6_S22_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dSI6_S22_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dSI6_S22_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dSI6_S22_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dSI6_S22_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dSI6_S22_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dSI6_S22_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dSI6_S22_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dSI6_S22_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dSI6_S22_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dSI6_S22_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dSI6_S22_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dSI6_S22_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dSI7_S23_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dSI7_S23_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dSI7_S23_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dSI7_S23_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dSI7_S23_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dSI7_S23_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dSI7_S23_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dSI7_S23_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dSI7_S23_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dSI7_S23_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dSI7_S23_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dSI7_S23_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dSI7_S23_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dSI7_S23_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dSI7_S23_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dSI7_S23_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dSI7_S23_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dSI7_S23_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dSI7_S23_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dSI7_S23_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dSI7_S23_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dSI7_S23_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dSI7_S23_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dSI7_S23_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dSI7_S23_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dSI7_S23_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dSI7_S23_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dSI7_S23_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dSI7_S23_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dSI7_S23_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dSI7_S23_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dSI7_S23_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dSI7_S23_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dSI7_S23_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dSI7_S23_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dSI7_S23_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dSI7_S23_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dSI7_S23_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dSI7_S23_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dSI7_S23_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dSI7_S23_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dSI7_S23_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dSI8_S24_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dSI8_S24_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dSI8_S24_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dSI8_S24_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dSI8_S24_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dSI8_S24_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dSI8_S24_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dSI8_S24_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dSI8_S24_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dSI8_S24_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dSI8_S24_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dSI8_S24_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dSI8_S24_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dSI8_S24_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dSI8_S24_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dSI8_S24_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dSI8_S24_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dSI8_S24_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dSI8_S24_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dSI8_S24_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dSI8_S24_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dSI8_S24_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dSI8_S24_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dSI8_S24_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dSI8_S24_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dSI8_S24_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dSI8_S24_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for 5dSI8_S24_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dSI8_S24_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dSI8_S24_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dSI8_S24_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dSI8_S24_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dSI8_S24_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dSI8_S24_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dSI8_S24_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dSI8_S24_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dSI8_S24_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dSI8_S24_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dSI8_S24_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dSI8_S24_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dSI8_S24_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dSI8_S24_L001_R2_001.fastq.gz\n",
      "Started analysis of 5dSI9_S25_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for 5dSI9_S25_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for 5dSI9_S25_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for 5dSI9_S25_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for 5dSI9_S25_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for 5dSI9_S25_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for 5dSI9_S25_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for 5dSI9_S25_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for 5dSI9_S25_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for 5dSI9_S25_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for 5dSI9_S25_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for 5dSI9_S25_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for 5dSI9_S25_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for 5dSI9_S25_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for 5dSI9_S25_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for 5dSI9_S25_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for 5dSI9_S25_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for 5dSI9_S25_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for 5dSI9_S25_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for 5dSI9_S25_L001_R1_001.fastq.gz\n",
      "Analysis complete for 5dSI9_S25_L001_R1_001.fastq.gz\n",
      "Started analysis of 5dSI9_S25_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for 5dSI9_S25_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for 5dSI9_S25_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for 5dSI9_S25_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for 5dSI9_S25_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for 5dSI9_S25_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 30% complete for 5dSI9_S25_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for 5dSI9_S25_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for 5dSI9_S25_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for 5dSI9_S25_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for 5dSI9_S25_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for 5dSI9_S25_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for 5dSI9_S25_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for 5dSI9_S25_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for 5dSI9_S25_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for 5dSI9_S25_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for 5dSI9_S25_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for 5dSI9_S25_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for 5dSI9_S25_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for 5dSI9_S25_L001_R2_001.fastq.gz\n",
      "Analysis complete for 5dSI9_S25_L001_R2_001.fastq.gz\n",
      "Started analysis of Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 5% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 10% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 15% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 20% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 25% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 30% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 35% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 40% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 45% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 50% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 55% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 60% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 65% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 70% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 75% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 80% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 85% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 90% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Approx 95% complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Analysis complete for Undetermined_S0_L001_R1_001.fastq.gz\n",
      "Started analysis of Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 5% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 10% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 15% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 20% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 25% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 30% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 35% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 40% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 45% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 50% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 55% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 60% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 65% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 70% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 75% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 80% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 85% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 90% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Approx 95% complete for Undetermined_S0_L001_R2_001.fastq.gz\n",
      "Analysis complete for Undetermined_S0_L001_R2_001.fastq.gz\n"
     ]
    }
   ],
   "source": [
    "#should this be done in bsub \"\" ?\n",
    "module load FastQC;\n",
    "\n",
    "fastqc \\\n",
    "$RAW/*.fastq.gz \\\n",
    "-o $FASTQC;\n",
    "\n",
    "module unload FastQC;"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[INFO   ]         multiqc : This is MultiQC v1.0.dev0\n",
      "[INFO   ]         multiqc : Template    : default\n",
      "[INFO   ]         multiqc : Searching '/workspace/Hrrmat/QIIME2/MGS7R1SeasonalCarbohydrate/010.fastqc'\n",
      "[INFO   ]          fastqc : Found 152 reports\n",
      "[INFO   ]         multiqc : Report      : ../../../workspace/Hrrmat/QIIME2/MGS7R1SeasonalCarbohydrate/020.multiqc/multiqc_report.html\n",
      "[INFO   ]         multiqc : Data        : ../../../workspace/Hrrmat/QIIME2/MGS7R1SeasonalCarbohydrate/020.multiqc/multiqc_data\n",
      "[INFO   ]         multiqc : MultiQC complete\n"
     ]
    }
   ],
   "source": [
    "# We run MultiQC on the outputs of FastQC, so that we can view a \n",
    "# combined report (rather than an individual FastQC report for each file)\n",
    "module unload qiime2;\n",
    "module load MultiQC;\n",
    "\n",
    "multiqc \\\n",
    "$FASTQC \\\n",
    "-o $MULTIQC;\n",
    "\n",
    "module unload MultiQC;"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "http://storage.powerplant.pfr.co.nz/workspace/Hrrmat/QIIME2/MGS7R1SeasonalCarbohydrate/020.multiqc/multiqc_report.html\n"
     ]
    }
   ],
   "source": [
    "# We create a link which you can click on to directly view the MultiQC report in a browser.\n",
    "echo http://storage.powerplant.pfr.co.nz$MULTIQC/multiqc_report.html"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "##### 3b. Assess multiQC report\n",
    "We can see some important points from the MultiQC report and the output generated while producing the report."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 4. Prepare sample metadata file and fastq manifest file\n",
    "Two text files are required for further analysis:\n",
    "\n",
    "1. A sample metadata file that includes (at a minimum) the name and a text description of the sample. The sample metadata file also includes barcodes and linker sequences, but if your files have already been demultiplexed, the barcode and linker sequence columns can be left blank. If your files have not been demultiplexed, QIIME2 can demultiplex them for you, but this is not required in this case. The sample metadata file also includes any metadata (any data that can be associated with a sample).\n",
    "\n",
    "2. A FASTQ manifest file that details the full file path of all the FASTQ files. This file must contain full paths to the files, and a description of whether they are forward or reverse reads (since this data is paired-end).\n",
    "\n",
    "In this case, we automatically generate both files. We create a list of all the left and right FASTQ files, and use various text search and manipulation programs ('awk', 'grep') to write the files. The sample names are taken from the start of the FASTQ file name (basically, the sample ID is taken from the text before the first underscore of the FASTQ file names). So, for the files\n",
    "\n",
    "2435-007_S76_L001_R1_001.fastq\n",
    "2435-007_S76_L001_R2_001.fastq\n",
    "which are a sample (2 paired files per sample), the sample name becomes\n",
    "\n",
    "2435-007\n",
    "This can be manually changed if required."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "##### 4b. Prepare the fastq manifest file¶\n",
    "This section is only required the first time that a fastq manifest file is required. In this case it is not required a second time so is commented out using #. Michelle manually prepared a fastq manifest file (/powerplant/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/fastq_manifest.csv) for Manuka from the complete NGS run R1 fastq manifest file (that included seasonal and Manuka data) by removing the unrequired .qz files. These included the seasonal files and \"Undetermined_S0_L001_R1_001.fastq.gz\" and \"Undetermined_S0_L001_R2_001.fastq.gz\" files. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 73,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "# Remove existing fastq manifest files to avoid confusion\n",
    "\n",
    "#rm -f $WORKING/fastq_manifest.csv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 74,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "# Write the header line\n",
    "#echo \"sample-id,absolute-filepath,direction\" > $WORKING/fastq_manifest.csv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 75,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "# Define a list of the forward and reverse reads\n",
    "#FORWARDREADS=$(ls $INPUT|grep R1|tr \"\\n\" \" \");\n",
    "#REVERSEREADS=$(ls $INPUT|grep R2|tr \"\\n\" \" \")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 76,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "# Use the defined list to grab a sample name (we use the start of the FASTQ filename before the first underscore),\n",
    "# the location and the direction, and append this to the existing manifest file\n",
    "# for READ in $FORWARDREADS\n",
    "# do\n",
    "# SAMPLENUM=$(echo $READ| awk 'BEGIN { FS = \"_\" } ; { print $1 }')\n",
    "# echo $SAMPLENUM,$RAW/$READ,forward >> $WORKING/fastq_manifest.csv\n",
    "# done\n",
    "\n",
    "# for READ in $REVERSEREADS\n",
    "# do\n",
    "# SAMPLENUM=$(echo $READ| awk 'BEGIN { FS = \"_\" } ; { print $1 }')\n",
    "# echo $SAMPLENUM,$RAW/$READ,reverse >> $WORKING/fastq_manifest.csv\n",
    "# done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "# If symlink error, fix by replacing file paths with physical file paths\n",
    "\n",
    "perl -i.bak -pne \"s|$RAW|$INPUT|g\" $WORKING/fastq_manifest.csv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "sample-id,absolute-filepath,direction\n",
      "5dDHA1,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dDHA1_S59_L001_R1_001.fastq.gz,forward\n",
      "5dDHA2,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dDHA2_S60_L001_R1_001.fastq.gz,forward\n",
      "5dDHA3,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dDHA3_S61_L001_R1_001.fastq.gz,forward\n",
      "5dDHA4,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dDHA4_S62_L001_R1_001.fastq.gz,forward\n",
      "5dDHA5,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dDHA5_S63_L001_R1_001.fastq.gz,forward\n",
      "5dDHA6,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dDHA6_S64_L001_R1_001.fastq.gz,forward\n",
      "5dDHA7,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dDHA7_S65_L001_R1_001.fastq.gz,forward\n",
      "5dDHA8,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dDHA8_S66_L001_R1_001.fastq.gz,forward\n",
      "5dDHA9,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dDHA9_S67_L001_R1_001.fastq.gz,forward\n",
      "5dH1,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dH1_S28_L001_R1_001.fastq.gz,forward\n",
      "5dH2,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dH2_S29_L001_R1_001.fastq.gz,forward\n",
      "5dH3,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dH3_S30_L001_R1_001.fastq.gz,forward\n",
      "5dH4,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dH4_S26_L001_R1_001.fastq.gz,forward\n",
      "5dH5,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dH5_S27_L001_R1_001.fastq.gz,forward\n",
      "5dIS1,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dIS1_S68_L001_R1_001.fastq.gz,forward\n",
      "5dIS2,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dIS2_S69_L001_R1_001.fastq.gz,forward\n",
      "5dIS3,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dIS3_S70_L001_R1_001.fastq.gz,forward\n",
      "5dIS4,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dIS4_S71_L001_R1_001.fastq.gz,forward\n",
      "5dIS6,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dIS6_S72_L001_R1_001.fastq.gz,forward\n",
      "5dIS7,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dIS7_S73_L001_R1_001.fastq.gz,forward\n",
      "5dIS8,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dIS8_S74_L001_R1_001.fastq.gz,forward\n",
      "5dIS9,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dIS9_S75_L001_R1_001.fastq.gz,forward\n",
      "5dMGO1,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMGO1_S51_L001_R1_001.fastq.gz,forward\n",
      "5dMGO2,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMGO2_S52_L001_R1_001.fastq.gz,forward\n",
      "5dMGO3,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMGO3_S53_L001_R1_001.fastq.gz,forward\n",
      "5dMGO4,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMGO4_S54_L001_R1_001.fastq.gz,forward\n",
      "5dMGO6,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMGO6_S55_L001_R1_001.fastq.gz,forward\n",
      "5dMGO7,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMGO7_S56_L001_R1_001.fastq.gz,forward\n",
      "5dMGO8,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMGO8_S57_L001_R1_001.fastq.gz,forward\n",
      "5dMGO9,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMGO9_S58_L001_R1_001.fastq.gz,forward\n",
      "5dMH15-1,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMH15-1_S35_L001_R1_001.fastq.gz,forward\n",
      "5dMH15-2,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMH15-2_S36_L001_R1_001.fastq.gz,forward\n",
      "5dMH15-3,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMH15-3_S37_L001_R1_001.fastq.gz,forward\n",
      "5dMH15-4,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMH15-4_S38_L001_R1_001.fastq.gz,forward\n",
      "5dMH15-6,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMH15-6_S39_L001_R1_001.fastq.gz,forward\n",
      "5dMH15-7,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMH15-7_S40_L001_R1_001.fastq.gz,forward\n",
      "5dMH15-8,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMH15-8_S41_L001_R1_001.fastq.gz,forward\n",
      "5dMH15-9,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMH15-9_S42_L001_R1_001.fastq.gz,forward\n",
      "5dMH17-1,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMH17-1_S43_L001_R1_001.fastq.gz,forward\n",
      "5dMH17-2,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMH17-2_S44_L001_R1_001.fastq.gz,forward\n",
      "5dMH17-3,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMH17-3_S45_L001_R1_001.fastq.gz,forward\n",
      "5dMH17-4,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMH17-4_S46_L001_R1_001.fastq.gz,forward\n",
      "5dMH17-6,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMH17-6_S47_L001_R1_001.fastq.gz,forward\n",
      "5dMH17-7,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMH17-7_S48_L001_R1_001.fastq.gz,forward\n",
      "5dMH17-8,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMH17-8_S49_L001_R1_001.fastq.gz,forward\n",
      "5dMH17-9,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMH17-9_S50_L001_R1_001.fastq.gz,forward\n",
      "5dSI1,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dSI1_S31_L001_R1_001.fastq.gz,forward\n",
      "5dSI2,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dSI2_S32_L001_R1_001.fastq.gz,forward\n",
      "5dSI3,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dSI3_S33_L001_R1_001.fastq.gz,forward\n",
      "5dSI4,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dSI4_S34_L001_R1_001.fastq.gz,forward\n",
      "5dSI5,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dSI5_S21_L001_R1_001.fastq.gz,forward\n",
      "5dSI6,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dSI6_S22_L001_R1_001.fastq.gz,forward\n",
      "5dSI7,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dSI7_S23_L001_R1_001.fastq.gz,forward\n",
      "5dSI8,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dSI8_S24_L001_R1_001.fastq.gz,forward\n",
      "5dSI9,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dSI9_S25_L001_R1_001.fastq.gz,forward\n",
      "5dDHA1,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dDHA1_S59_L001_R2_001.fastq.gz,reverse\n",
      "5dDHA2,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dDHA2_S60_L001_R2_001.fastq.gz,reverse\n",
      "5dDHA3,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dDHA3_S61_L001_R2_001.fastq.gz,reverse\n",
      "5dDHA4,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dDHA4_S62_L001_R2_001.fastq.gz,reverse\n",
      "5dDHA5,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dDHA5_S63_L001_R2_001.fastq.gz,reverse\n",
      "5dDHA6,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dDHA6_S64_L001_R2_001.fastq.gz,reverse\n",
      "5dDHA7,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dDHA7_S65_L001_R2_001.fastq.gz,reverse\n",
      "5dDHA8,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dDHA8_S66_L001_R2_001.fastq.gz,reverse\n",
      "5dDHA9,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dDHA9_S67_L001_R2_001.fastq.gz,reverse\n",
      "5dH1,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dH1_S28_L001_R2_001.fastq.gz,reverse\n",
      "5dH2,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dH2_S29_L001_R2_001.fastq.gz,reverse\n",
      "5dH3,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dH3_S30_L001_R2_001.fastq.gz,reverse\n",
      "5dH4,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dH4_S26_L001_R2_001.fastq.gz,reverse\n",
      "5dH5,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dH5_S27_L001_R2_001.fastq.gz,reverse\n",
      "5dIS1,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dIS1_S68_L001_R2_001.fastq.gz,reverse\n",
      "5dIS2,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dIS2_S69_L001_R2_001.fastq.gz,reverse\n",
      "5dIS3,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dIS3_S70_L001_R2_001.fastq.gz,reverse\n",
      "5dIS4,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dIS4_S71_L001_R2_001.fastq.gz,reverse\n",
      "5dIS6,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dIS6_S72_L001_R2_001.fastq.gz,reverse\n",
      "5dIS7,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dIS7_S73_L001_R2_001.fastq.gz,reverse\n",
      "5dIS8,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dIS8_S74_L001_R2_001.fastq.gz,reverse\n",
      "5dIS9,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dIS9_S75_L001_R2_001.fastq.gz,reverse\n",
      "5dMGO1,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMGO1_S51_L001_R2_001.fastq.gz,reverse\n",
      "5dMGO2,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMGO2_S52_L001_R2_001.fastq.gz,reverse\n",
      "5dMGO3,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMGO3_S53_L001_R2_001.fastq.gz,reverse\n",
      "5dMGO4,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMGO4_S54_L001_R2_001.fastq.gz,reverse\n",
      "5dMGO6,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMGO6_S55_L001_R2_001.fastq.gz,reverse\n",
      "5dMGO7,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMGO7_S56_L001_R2_001.fastq.gz,reverse\n",
      "5dMGO8,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMGO8_S57_L001_R2_001.fastq.gz,reverse\n",
      "5dMGO9,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMGO9_S58_L001_R2_001.fastq.gz,reverse\n",
      "5dMH15-1,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMH15-1_S35_L001_R2_001.fastq.gz,reverse\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "5dMH15-2,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMH15-2_S36_L001_R2_001.fastq.gz,reverse\n",
      "5dMH15-3,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMH15-3_S37_L001_R2_001.fastq.gz,reverse\n",
      "5dMH15-4,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMH15-4_S38_L001_R2_001.fastq.gz,reverse\n",
      "5dMH15-6,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMH15-6_S39_L001_R2_001.fastq.gz,reverse\n",
      "5dMH15-7,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMH15-7_S40_L001_R2_001.fastq.gz,reverse\n",
      "5dMH15-8,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMH15-8_S41_L001_R2_001.fastq.gz,reverse\n",
      "5dMH15-9,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMH15-9_S42_L001_R2_001.fastq.gz,reverse\n",
      "5dMH17-1,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMH17-1_S43_L001_R2_001.fastq.gz,reverse\n",
      "5dMH17-2,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMH17-2_S44_L001_R2_001.fastq.gz,reverse\n",
      "5dMH17-3,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMH17-3_S45_L001_R2_001.fastq.gz,reverse\n",
      "5dMH17-4,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMH17-4_S46_L001_R2_001.fastq.gz,reverse\n",
      "5dMH17-6,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMH17-6_S47_L001_R2_001.fastq.gz,reverse\n",
      "5dMH17-7,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMH17-7_S48_L001_R2_001.fastq.gz,reverse\n",
      "5dMH17-8,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMH17-8_S49_L001_R2_001.fastq.gz,reverse\n",
      "5dMH17-9,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dMH17-9_S50_L001_R2_001.fastq.gz,reverse\n",
      "5dSI1,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dSI1_S31_L001_R2_001.fastq.gz,reverse\n",
      "5dSI2,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dSI2_S32_L001_R2_001.fastq.gz,reverse\n",
      "5dSI3,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dSI3_S33_L001_R2_001.fastq.gz,reverse\n",
      "5dSI4,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dSI4_S34_L001_R2_001.fastq.gz,reverse\n",
      "5dSI5,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dSI5_S21_L001_R2_001.fastq.gz,reverse\n",
      "5dSI6,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dSI6_S22_L001_R2_001.fastq.gz,reverse\n",
      "5dSI7,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dSI7_S23_L001_R2_001.fastq.gz,reverse\n",
      "5dSI8,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dSI8_S24_L001_R2_001.fastq.gz,reverse\n",
      "5dSI9,/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/000.raw/5dSI9_S25_L001_R2_001.fastq.gz,reverse\n"
     ]
    }
   ],
   "source": [
    "# View the FASTQ manifest file to make sure it looks OK\n",
    "cat $WORKING/fastq_manifest.csv"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "We can see that the FASTQ manifest file is a .csv (comma separated value) file with three columns (the sample Id, the absolute filepath, and the direction of the read (forward or reverse). Sample Psa2 is removed since we have determined it has no reads. All \"forward\" files must be matched by a \"reverse\" file."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### 4c. Do not need to Prepare a sample metadata file"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "# Remove existing fastq manifest files to avoid confusion\n",
    "\n",
    "rm -f $WORKING/sample_metadata.tsv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "# Write the header line\n",
    "echo -e \"#SampleID\\tBarcodeSequence\\tLinkerPrimerSequence\\tMethods\\tDescription\" >> $WORKING/sample_metadata.tsv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "for READ in $FORWARDREADS\n",
    "do\n",
    "SAMPLENUM=$(echo $READ| awk 'BEGIN { FS = \"_\" } ; { print $1 }')\n",
    "echo -e \"$SAMPLENUM\\t\\t\\t$SAMPLENUM\" >> $WORKING/sample_metadata.tsv\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "#SampleID\tBarcodeSequence\tLinkerPrimerSequence\tMethods\tTrial\tTreatment\tDHA_mgkg\tMG_mgkg\tHMF_mgkg\tNPA\tPercent_Mortality\n",
      "5dDHA1\t\t\t5dDHA1\t5d\tDHA\t1692\t0\t0\t0\t0\n",
      "5dDHA2\t\t\t5dDHA2\t5d\tDHA\t1692\t0\t0\t0\t0\n",
      "5dDHA3\t\t\t5dDHA3\t5d\tDHA\t1692\t0\t0\t0\t0\n",
      "5dDHA4\t\t\t5dDHA4\t5d\tDHA\t1692\t0\t0\t0\t0\n",
      "5dDHA5\t\t\t5dDHA5\t5d\tDHA\t1692\t0\t0\t0\t0\n",
      "5dDHA6\t\t\t5dDHA6\t5d\tDHA\t1692\t0\t0\t0\t0\n",
      "5dDHA7\t\t\t5dDHA7\t5d\tDHA\t1692\t0\t0\t0\t0\n",
      "5dDHA8\t\t\t5dDHA8\t5d\tDHA\t1692\t0\t0\t0\t0\n",
      "5dDHA9\t\t\t5dDHA9\t5d\tDHA\t1692\t0\t0\t0\t0\n",
      "5dH1\t\t\t5dH1\t5d\tHive\t0\t0\t0\t0\t0\n",
      "5dH2\t\t\t5dH2\t5d\tHive\t0\t0\t0\t0\t0\n",
      "5dH3\t\t\t5dH3\t5d\tHive\t0\t0\t0\t0\t0\n",
      "5dH4\t\t\t5dH4\t5d\tHive\t0\t0\t0\t0\t0\n",
      "5dH5\t\t\t5dH5\t5d\tHive\t0\t0\t0\t0\t0\n",
      "5dIS1\t\t\t5dIS1\t5d\tInvertSugar\t0\t0\t0\t0\t0\n",
      "5dIS2\t\t\t5dIS2\t5d\tInvertSugar\t0\t0\t0\t0\t0\n",
      "5dIS3\t\t\t5dIS3\t5d\tInvertSugar\t0\t0\t0\t0\t0\n",
      "5dIS4\t\t\t5dIS4\t5d\tInvertSugar\t0\t0\t0\t0\t0\n",
      "5dIS6\t\t\t5dIS6\t5d\tInvertSugar\t0\t0\t0\t0\t0\n",
      "5dIS7\t\t\t5dIS7\t5d\tInvertSugar\t0\t0\t0\t0\t0\n",
      "5dIS8\t\t\t5dIS8\t5d\tInvertSugar\t0\t0\t0\t0\t0\n",
      "5dIS9\t\t\t5dIS9\t5d\tInvertSugar\t0\t0\t0\t0\t0\n",
      "5dMGO1\t\t\t5dMGO1\t5d\tMGO\t0\t745\t0\t0\t0\n",
      "5dMGO2\t\t\t5dMGO2\t5d\tMGO\t0\t745\t0\t0\t0\n",
      "5dMGO3\t\t\t5dMGO3\t5d\tMGO\t0\t745\t0\t0\t0\n",
      "5dMGO4\t\t\t5dMGO4\t5d\tMGO\t0\t745\t0\t0\t0\n",
      "5dMGO6\t\t\t5dMGO6\t5d\tMGO\t0\t745\t0\t0\t0\n",
      "5dMGO7\t\t\t5dMGO7\t5d\tMGO\t0\t745\t0\t0\t0\n",
      "5dMGO8\t\t\t5dMGO8\t5d\tMGO\t0\t745\t0\t0\t0\n",
      "5dMGO9\t\t\t5dMGO9\t5d\tMGO\t0\t745\t0\t0\t0\n",
      "5dMH15-1\t\t\t5dMH15-1\t5d\tMH2015\t1238\t744.6\t11.9\t18.7\t0\n",
      "5dMH15-2\t\t\t5dMH15-2\t5d\tMH2015\t1238\t744.6\t11.9\t18.7\t0\n",
      "5dMH15-3\t\t\t5dMH15-3\t5d\tMH2015\t1238\t744.6\t11.9\t18.7\t0\n",
      "5dMH15-4\t\t\t5dMH15-4\t5d\tMH2015\t1238\t744.6\t11.9\t18.7\t0\n",
      "5dMH15-6\t\t\t5dMH15-6\t5d\tMH2015\t1238\t744.6\t11.9\t18.7\t0\n",
      "5dMH15-7\t\t\t5dMH15-7\t5d\tMH2015\t1238\t744.6\t11.9\t18.7\t0\n",
      "5dMH15-8\t\t\t5dMH15-8\t5d\tMH2015\t1238\t744.6\t11.9\t18.7\t0\n",
      "5dMH15-9\t\t\t5dMH15-9\t5d\tMH2015\t1238\t744.6\t11.9\t18.7\t0\n",
      "5dMH17-1\t\t\t5dMH17-1\t5d\tMH2017\t1691\t393.9\t3.5\t12.8\t0\n",
      "5dMH17-2\t\t\t5dMH17-2\t5d\tMH2017\t1691\t393.9\t3.5\t12.8\t0\n",
      "5dMH17-3\t\t\t5dMH17-3\t5d\tMH2017\t1691\t393.9\t3.5\t12.8\t0\n",
      "5dMH17-4\t\t\t5dMH17-4\t5d\tMH2017\t1691\t393.9\t3.5\t12.8\t0\n",
      "5dMH17-6\t\t\t5dMH17-6\t5d\tMH2017\t1691\t393.9\t3.5\t12.8\t0\n",
      "5dMH17-7\t\t\t5dMH17-7\t5d\tMH2017\t1691\t393.9\t3.5\t12.8\t0\n",
      "5dMH17-8\t\t\t5dMH17-8\t5d\tMH2017\t1691\t393.9\t3.5\t12.8\t0\n",
      "5dMH17-9\t\t\t5dMH17-9\t5d\tMH2017\t1691\t393.9\t3.5\t12.8\t0\n",
      "5dSI1\t\t\t5dSI1\t5d\tSucrose\t0\t0\t0\t0\t0\n",
      "5dSI2\t\t\t5dSI2\t5d\tSucrose\t0\t0\t0\t0\t0\n",
      "5dSI3\t\t\t5dSI3\t5d\tSucrose\t0\t0\t0\t0\t0\n",
      "5dSI4\t\t\t5dSI4\t5d\tSucrose\t0\t0\t0\t0\t0\n",
      "5dSI5\t\t\t5dSI5\t5d\tSucrose\t0\t0\t0\t0\t0\n",
      "5dSI6\t\t\t5dSI6\t5d\tSucrose\t0\t0\t0\t0\t0\n",
      "5dSI7\t\t\t5dSI7\t5d\tSucrose\t0\t0\t0\t0\t0\n",
      "5dSI8\t\t\t5dSI8\t5d\tSucrose\t0\t0\t0\t0\t0\n",
      "5dSI9\t\t\t5dSI9\t5d\tSucrose\t0\t0\t0\t0\t0\n"
     ]
    }
   ],
   "source": [
    "# Check the file in Excel and make changes then save to WinSCP\n",
    "cat $WORKING/metadata5dManuka.tsv"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "We can see that the sample metadata file is a .tsv (tab separated value) file with 4 mandatory columns and 1 optional column.The SampleID must match the sample-id in the FASTQ manifest file. In this case, the LinkerPrimerSequence column has been left blank, but if you have data that is not demultiplexed, including the BarcodeSequence and LinkerPrimerSequence in this file will allow QIIME2 to demultiplex for you. In this case the metadata is about the treatment dose fed to the bees, but it can be anything.\n",
    "\n",
    "Including metadata allows downstream steps to determine whether variation in bacterial profiles are associated with a particular metadata trait (e.g. extraction method, location, time, treatment)."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Now we are ready to run QIIME2. QIIME2 imports your FASTQ files into a special type of QIIME2 file, and produces various other QIIME2 files throughout the analysis. There are two main file types:\n",
    "\n",
    "    .qza files: QIIME2 data files that are designed to be used in various analysis steps\n",
    "    .qzv files: QIIME2 visualisation files that are designed to be viewed in the QIIME2 viewer.\n",
    "\n",
    "It's possible to view both .qza and .qzv files in the QIIME2 viewer. Viewing .qza files will give some basic information on the files. However viewing .qzv files in the QIIME2 viewer is the main way of viewing and interpreting the analysis."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 5. Import your FASTQ files (with appropriate metadata) into a QIIME2 .qza file¶"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "First, we set up QIIME2. First, we must do a small hack that allows QIIME2 to produce displayable .qzv files without actually automatically trying to display them (since we simply want to produce the files and view them later). This hack is discussed here. Note that this changes your matplotlib config file."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "echo \"backend: Agg\" > ~/.config/matplotlib/matplotlibrc"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "module unload qiime2/2018.2;\n",
    "module load qiime2/2018.2;"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Now let's use qiime tools import to import our FASTQ files into a .qza file."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 82,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "qiime tools import \\\n",
    "  --type SampleData[PairedEndSequencesWithQuality] \\\n",
    "  --input-path $WORKING/fastq_manifest.csv \\\n",
    "  --output-path $IMPORT/Paired_end_demux.qza \\\n",
    "  --source-format PairedEndFastqManifestPhred33"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Let's examine the contents of IMPORT to ensure there's a file called Paired_end_demux.qza of appropriate size in there."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 128,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "total 3129939\n",
      "3129370 Paired_end_demux.qza      570 Paired_end_demux.qzv\n"
     ]
    }
   ],
   "source": [
    "ls -s $IMPORT"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Now let's go through our first example of producing and viewing a .qzv QIIME2 visualisation file. We are running a program called demux summarise which summarises and graphically presents the sequence information contained within the Paired_end_demux.qza file. We will produce the file, download it, and drag-and-drop into the external site https://view.qiime2.org/ (consider data security here). There are other options for viewing .qzv files locally and for exporting the graphics to other formats."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 84,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: /workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/030.import/Paired_end_demux.qzv\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "# Run demux summarise to produce viewing file\n",
    "qiime demux summarize \\\n",
    "  --i-data $IMPORT/Paired_end_demux.qza \\\n",
    "  --o-visualization $IMPORT/Paired_end_demux.qzv"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "You can see that QIIME2 tells you when a new file has successfully been created. Let's download the file by creating a link and then clicking on it."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 85,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "http://storage.powerplant.pfr.co.nz/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/030.import/Paired_end_demux.qzv\n"
     ]
    }
   ],
   "source": [
    "# We create a link which you can click on to download the file Paired_end_demux.qzv.\n",
    "echo http://storage.powerplant.pfr.co.nz$IMPORT/Paired_end_demux.qzv"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Now, click on https://view.qiime2.org/ to open the site in a new tab. Then drag-and-drop the file Paired_end_demux.qzv into the drag-and-drop area to view the file. You should see an Overview and an Interactive Quality Plot."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 6. Filtering and Chimera removal with DADA2\n",
    "We use dada2 denoise-paired to remove artefacts in paired-end Illumina data. From the QIIME2 tutorial\n",
    "\n",
    "DADA2 is a pipeline for detecting and correcting (where possible) Illumina amplicon sequence data. As implemented in the q2-dada2 plugin, this quality control process will additionally filter any phiX reads (commonly present in marker gene Illumina sequence data) that are identified in the sequencing data, and will filter chimeric sequences.\n",
    "\n",
    "Note that we have set --p-trim-left-f \\ --p-trim-left-r to 7 (which will trim off the first 7 bases) and --p-trunc-len-f \\ --p-trunc-len-r to 245 (which will truncate the ~250bp reads to 245).\n",
    "\n",
    "This is the longest step in the process, taking ~60 minutes upto a couple of days (approx). We set the --verbose switch to show more detailed output on what's happening. You will see some warning messages (in this case, don't worry).\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 86,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Job <205628> is submitted to default queue <normal>.\n"
     ]
    }
   ],
   "source": [
    "bsub \"qiime dada2 denoise-paired \\\n",
    "  --i-demultiplexed-seqs $IMPORT/Paired_end_demux.qza \\\n",
    "  --p-trim-left-f 7 \\\n",
    "  --p-trim-left-r 7 \\\n",
    "  --p-trunc-len-f 245 \\\n",
    "  --p-trunc-len-r 245 \\\n",
    "  --verbose \\\n",
    "  --o-representative-sequences $FILTER/rep-seqs-dada2.qza \\\n",
    "  --o-table $FILTER/table-dada2.qza\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 88,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "JOBID   USER    STAT  QUEUE      FROM_HOST   EXEC_HOST   JOB_NAME   SUBMIT_TIME\n",
      "205628  hrrmat  DONE  normal     aklppr31    wkoppb37    *ences /wo May  9 14:59\n"
     ]
    }
   ],
   "source": [
    "bjobs 205628\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<< output from stdout >>\n",
      "R version 3.3.1 (2016-06-21) \n",
      "Loading required package: Rcpp\n",
      "Warning messages:\n",
      "1: multiple methods tables found for ‘arbind’ \n",
      "2: multiple methods tables found for ‘acbind’ \n",
      "3: replacing previous import ‘IRanges::arbind’ by ‘SummarizedExperiment::arbind’ when loading ‘GenomicAlignments’ \n",
      "4: replacing previous import ‘IRanges::acbind’ by ‘SummarizedExperiment::acbind’ when loading ‘GenomicAlignments’ \n",
      "5: multiple methods tables found for ‘left’ \n",
      "6: multiple methods tables found for ‘right’ \n",
      "DADA2 R package version: 1.4.0 \n",
      "1) Filtering ............................................................................\n",
      "2) Learning Error Rates\n",
      "2a) Forward Reads\n",
      "Initializing error rates to maximum possible estimate.\n",
      "Sample 1 - 92533 reads in 10808 unique sequences.\n",
      "Sample 2 - 82201 reads in 10012 unique sequences.\n",
      "Sample 3 - 98587 reads in 13642 unique sequences.\n",
      "Sample 4 - 98577 reads in 13487 unique sequences.\n",
      "Sample 5 - 100019 reads in 12588 unique sequences.\n",
      "Sample 6 - 118536 reads in 15270 unique sequences.\n",
      "Sample 7 - 102229 reads in 14826 unique sequences.\n",
      "Sample 8 - 100726 reads in 13686 unique sequences.\n",
      "Sample 9 - 110770 reads in 14076 unique sequences.\n",
      "Sample 10 - 94127 reads in 11246 unique sequences.\n",
      "Sample 11 - 100165 reads in 13468 unique sequences.\n",
      "   selfConsist step 2 \n",
      "   selfConsist step 3 \n",
      "   selfConsist step 4 \n",
      "   selfConsist step 5 \n",
      "\n",
      "\n",
      "Convergence after  5  rounds.\n",
      "2b) Reverse Reads\n",
      "Initializing error rates to maximum possible estimate.\n",
      "Sample 1 - 92533 reads in 45228 unique sequences.\n",
      "Sample 2 - 82201 reads in 41135 unique sequences.\n",
      "Sample 3 - 98587 reads in 43368 unique sequences.\n",
      "Sample 4 - 98577 reads in 42761 unique sequences.\n",
      "Sample 5 - 100019 reads in 42974 unique sequences.\n",
      "Sample 6 - 118536 reads in 50032 unique sequences.\n",
      "Sample 7 - 102229 reads in 46647 unique sequences.\n",
      "Sample 8 - 100726 reads in 43412 unique sequences.\n",
      "Sample 9 - 110770 reads in 48322 unique sequences.\n",
      "Sample 10 - 94127 reads in 43239 unique sequences.\n",
      "Sample 11 - 100165 reads in 43357 unique sequences.\n",
      "   selfConsist step 2 \n",
      "   selfConsist step 3 \n",
      "   selfConsist step 4 \n",
      "   selfConsist step 5 \n",
      "\n",
      "\n",
      "Convergence after  5  rounds.\n",
      "\n",
      "3) Denoise remaining samples .................................................................\n",
      "The sequences being tabled vary in length.\n",
      "4) Remove chimeras (method = consensus)\n",
      "tail: /home/hrrmat/.lsbatch/1525208121.195801.out: Stale file handle\n",
      "tail: no files remaining\n"
     ]
    }
   ],
   "source": [
    "bpeek -f 195801\n",
    "\n",
    "#(Insert the bjobs number from In 22 into the line above. 195539 is the job number. This changes every time), \n",
    "#It will take a long time… hours or days! Depending on the size of my data, \n",
    "#But since it’s running on the cluster it will run even if the notebook glitches or fails.\n",
    "\n",
    "#Once it’s done, you should be able to carry on with the analysis."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "bkill 187031"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 7. Examine features and sequences\n",
    "DADA2 produces tables of sequences and \"features\" (which are groups of similar sequences that may be from the same species or taxonomic unit). We use feature-table summarise and feature-table tabulate-seqs to produce .qzv files that we can use to explore the sequences and features. From the QIIME2 tutorial\n",
    "\n",
    "After the quality filtering step completes, you’ll want to explore the resulting data. You can do this using the following two commands, which will create visual summaries of the data. The feature-table summarize command will give you information on how many sequences are associated with each sample and with each feature, histograms of those distributions, and some related summary statistics. The feature-table tabulate-seqs command will provide a mapping of feature IDs to sequences, and provide links to easily BLAST each sequence against the NCBI nt database. The latter visualization will be very useful later in the tutorial, when you want to learn more about specific features that are important in the data set."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 89,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: /workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/050.otu/table.qzv\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "module load qiime2/2018.2;\n",
    "qiime feature-table summarize \\\n",
    "  --i-table $FILTER/table-dada2.qza \\\n",
    "  --o-visualization $OTU/table.qzv\n",
    "module unload qiime2/2018.2;"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 90,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: /workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/050.otu/rep-seqs.qzv\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "module load qiime2/2018.2;\n",
    "qiime feature-table tabulate-seqs \\\n",
    "  --i-data $FILTER/rep-seqs-dada2.qza \\\n",
    "  --o-visualization $OTU/rep-seqs.qzv \\\n",
    "  --verbose\n",
    "module unload qiime2/2018.2;"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Now let's examine the outputs table.qzv and rep-seqs.qzv."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 91,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "http://storage.powerplant.pfr.co.nz/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/050.otu/table.qzv\n",
      "http://storage.powerplant.pfr.co.nz/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/050.otu/rep-seqs.qzv\n"
     ]
    }
   ],
   "source": [
    "# We create a link which you can click on to download the files table.qzv and rep-seqs.qzv.\n",
    "echo http://storage.powerplant.pfr.co.nz$OTU/table.qzv\n",
    "echo http://storage.powerplant.pfr.co.nz$OTU/rep-seqs.qzv"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Now, click on https://view.qiime2.org/ to open the site in a new tab (do it twice, since you'll need two open tabs for the two files. Then drag-and-drop the files table.qzv and rep-seqs.qzv into the drag-and-drop area to view the file."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 8. Generate a phylogeny of the sequences\n",
    "Phylogenetic diversity (PD) measures the diversity of species in a sample, and is expressed as the number of tree units which are found in a sample.We use Faith (1992)'s measure of PD. http://dx.doi.org/10.1016/0006-3207(92)91201-3.\n",
    "\n",
    "The phylogeny show \"representative\" sequences from each feature (or OTU), so all features from all samples are included."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 92,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureData[AlignedSequence] to: /workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/060.phylogeny/aligned-rep-seqs.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "# First, we do a multiple sequence alignment of all representative sequences\n",
    "module load qiime2/2018.2;\n",
    "qiime alignment mafft \\\n",
    "  --i-sequences $FILTER/rep-seqs-dada2.qza \\\n",
    "  --o-alignment $PHYLOGENY/aligned-rep-seqs.qza\n",
    "module unload qiime2/2018.2;"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 93,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureData[AlignedSequence] to: /workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/060.phylogeny/masked-aligned-rep-seqs.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "# Next, we mask (or filter) the alignment to remove positions that are highly variable.\n",
    "# These positions are generally considered to add noise to a resulting phylogenetic tree.\n",
    "module load qiime2/2018.2;\n",
    "qiime alignment mask \\\n",
    "  --i-alignment $PHYLOGENY/aligned-rep-seqs.qza \\\n",
    "  --o-masked-alignment $PHYLOGENY/masked-aligned-rep-seqs.qza\n",
    "module unload qiime2/2018.2;"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 94,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Phylogeny[Unrooted] to: /workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/060.phylogeny/unrooted-tree.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "# Now we generate a tree using fasttree\n",
    "module load qiime2/2018.2;\n",
    "qiime phylogeny fasttree \\\n",
    "  --i-alignment $PHYLOGENY/masked-aligned-rep-seqs.qza \\\n",
    "  --o-tree $PHYLOGENY/unrooted-tree.qza\n",
    "module unload qiime2/2018.2;"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 95,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Phylogeny[Rooted] to: /workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/060.phylogeny/rooted-tree.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "# The FastTree program creates an unrooted tree, so \n",
    "# we apply midpoint rooting to place the root of the tree \n",
    "# at the midpoint of the longest tip-to-tip distance in the unrooted tree.\n",
    "module load qiime2/2018.2;\n",
    "qiime phylogeny midpoint-root \\\n",
    "  --i-tree $PHYLOGENY/unrooted-tree.qza \\\n",
    "  --o-rooted-tree $PHYLOGENY/rooted-tree.qza\n",
    "module unload qiime2/2018.2;"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 96,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "# Now,we export the tree to a \"newick\" format file in the PHYLOGENY directory\n",
    "module load qiime2/2018.2;\n",
    "qiime tools export \\\n",
    "$PHYLOGENY/rooted-tree.qza --output-dir $PHYLOGENY\n",
    "module unload qiime2/2018.2;"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 97,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "http://storage.powerplant.pfr.co.nz/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/060.phylogeny\n"
     ]
    }
   ],
   "source": [
    "# We create a link which you can click on to download the tree file\n",
    "echo http://storage.powerplant.pfr.co.nz$PHYLOGENY"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The tree can be viewed using Treeview; either cut and paste the text from the tree, or save the tree.nwk and import into Treeview."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 9. Alpha and Beta diversity¶\n",
    "\n",
    "#### https://docs.qiime2.org/2018.6/tutorials/moving-pictures/?highlight=unweighted%20unifrac"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "QIIME 2’s diversity analyses are available through the q2-diversity plugin, which supports computing alpha and beta diversity metrics, applying related statistical tests, and generating interactive visualizations. We’ll first apply the core-metrics-phylogenetic method, which rarefies a FeatureTable[Frequency] to a user-specified depth, computes several alpha and beta diversity metrics, and generates principle coordinates analysis (PCoA) plots using Emperor for each of the beta diversity metrics. The metrics computed by default are:\n",
    "\n",
    "### Alpha diversity: Diversity within a sample\n",
    "\n",
    "Shannon’s diversity index (a quantitative measure of community richness).\n",
    "\n",
    "Observed OTUs (a qualitative measure of community richness).\n",
    "\n",
    "Faith’s Phylogenetic Diversity (a qualitiative measure of community richness that incorporates phylogenetic relationships between the features).\n",
    "\n",
    "Evenness (or Pielou’s Evenness; a measure of community evenness).\n",
    "\n",
    "### Beta diversity: Diversity between samples\n",
    "\n",
    "Jaccard distance (a qualitative measure of community dissimilarity).\n",
    "\n",
    "Bray-Curtis distance (a quantitative measure of community dissimilarity).\n",
    "\n",
    "Unweighted UniFrac distance (a qualitative measure of community dissimilarity that incorporates phylogenetic relationships between the features).\n",
    "\n",
    "Weighted UniFrac distance (a quantitative measure of community dissimilarity that incorporates phylogenetic relationships between the features).\n",
    "\n",
    "#### --p-sampling-depth\n",
    "The parameter required for this script is --p-sampling-depth, which is the even sampling (i.e. rarefaction) depth. Because most diversity metrics are sensitive to different sampling depths across different samples, this script will randomly subsample the counts from each sample to the value provided for this parameter. For example, if you provide --p-sampling-depth 500, this step will subsample the counts in each sample without replacement so that each sample in the resulting table has a total count of 500. If the total count for any sample(s) are smaller than this value, those samples will be dropped from the diversity analysis. Choose the value by reviewing the information presented in the table.qzv file that was created above and choosing a value that is as high as possible (so you retain more sequences per sample) while excluding as few samples as possible."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 63,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mkdir: cannot create directory ‘/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/070.diversity_backup’: File exists\n"
     ]
    }
   ],
   "source": [
    "# For some reason (previous run) there are already outputs in the $DIVERSITY directory.\n",
    "# move these to a backup folder and delete\n",
    "mkdir ${DIVERSITY}_backup;\n",
    "mv ${DIVERSITY}/* ${DIVERSITY}_backup;"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "module load qiime2/2018.2;\n",
    "qiime diversity core-metrics-phylogenetic --help\n",
    "module unload qiime2/2018.2;"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 66,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: /workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/070.diversity/rarefied_table.qza\u001b[0m\n",
      "\u001b[32mSaved SampleData[AlphaDiversity] % Properties(['phylogenetic']) to: /workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/070.diversity/faith_pd_vector.qza\u001b[0m\n",
      "\u001b[32mSaved SampleData[AlphaDiversity] to: /workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/070.diversity/observed_otus_vector.qza\u001b[0m\n",
      "\u001b[32mSaved SampleData[AlphaDiversity] to: /workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/070.diversity/shannon_vector.qza\u001b[0m\n",
      "\u001b[32mSaved SampleData[AlphaDiversity] to: /workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/070.diversity/evenness_vector.qza\u001b[0m\n",
      "\u001b[32mSaved DistanceMatrix % Properties(['phylogenetic']) to: /workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/070.diversity/unweighted_unifrac_distance_matrix.qza\u001b[0m\n",
      "\u001b[32mSaved DistanceMatrix % Properties(['phylogenetic']) to: /workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/070.diversity/weighted_unifrac_distance_matrix.qza\u001b[0m\n",
      "\u001b[32mSaved DistanceMatrix to: /workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/070.diversity/jaccard_distance_matrix.qza\u001b[0m\n",
      "\u001b[32mSaved DistanceMatrix to: /workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/070.diversity/bray_curtis_distance_matrix.qza\u001b[0m\n",
      "\u001b[32mSaved PCoAResults to: /workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/070.diversity/unweighted_unifrac_pcoa_results.qza\u001b[0m\n",
      "\u001b[32mSaved PCoAResults to: /workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/070.diversity/weighted_unifrac_pcoa_results.qza\u001b[0m\n",
      "\u001b[32mSaved PCoAResults to: /workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/070.diversity/jaccard_pcoa_results.qza\u001b[0m\n",
      "\u001b[32mSaved PCoAResults to: /workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/070.diversity/bray_curtis_pcoa_results.qza\u001b[0m\n",
      "\u001b[32mSaved Visualization to: /workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/070.diversity/unweighted_unifrac_emperor.qzv\u001b[0m\n",
      "\u001b[32mSaved Visualization to: /workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/070.diversity/weighted_unifrac_emperor.qzv\u001b[0m\n",
      "\u001b[32mSaved Visualization to: /workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/070.diversity/jaccard_emperor.qzv\u001b[0m\n",
      "\u001b[32mSaved Visualization to: /workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/070.diversity/bray_curtis_emperor.qzv\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "# Since this program fails if the output directory already exists, remove it, as it will be recreated\n",
    "\n",
    "rmdir $DIVERSITY;\n",
    "#The --p-sampling-depth parameter was set to 58220 because this is the smallest number of sequences which was in sample 5dMH15-9\n",
    "#This will retain all of my samples. \n",
    "#Check in Section 7 which produces folder 050.otu with the file table.qzv for minimum frequency. \n",
    "#Copy to K drive then Drag and drop in view.qiime2.org.\n",
    "echo http://storage.powerplant.pfr.co.nz$OTU/table.qzv\n",
    "module load qiime2/2018.2;\n",
    "qiime diversity core-metrics-phylogenetic \\\n",
    "  --i-phylogeny $PHYLOGENY/rooted-tree.qza \\\n",
    "  --i-table $FILTER/table-dada2.qza \\\n",
    "  --p-sampling-depth 58220 \\\n",
    "  --m-metadata-file $WORKING/metadata5dManuka.tsv \\\n",
    "  --output-dir $DIVERSITY\n",
    "module unload qiime2/2018.2;\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Metadata\n",
    "Now explore the microbial composition of the samples in the context of the sample metadata. \n",
    "\n",
    "First test for associations between categorical metadata columns and alpha diversity data. We’ll do that here for the Faith Phylogenetic Diversity (a measure of community richness) and evenness metrics."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "module load qiime2/2018.2;\n",
    "qiime diversity alpha-group-significance --help\n",
    "module unload qiime2/2018.2;"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 70,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: /workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/070.diversity/faith-pd-group-significance.qzv\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "module load qiime2/2018.2;\n",
    "qiime diversity alpha-group-significance \\\n",
    "  --i-alpha-diversity $DIVERSITY/faith_pd_vector.qza \\\n",
    "  --m-metadata-file $WORKING/metadata5dManuka.tsv \\\n",
    "  --o-visualization $DIVERSITY/faith-pd-group-significance.qzv\n",
    "module unload qiime2/2018.2;"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 69,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: /workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/070.diversity/evenness-group-significance.qzv\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "module load qiime2/2018.2;\n",
    "qiime diversity alpha-group-significance \\\n",
    "  --i-alpha-diversity $DIVERSITY/evenness_vector.qza \\\n",
    "  --m-metadata-file $WORKING/metadata5dManuka.tsv \\\n",
    "  --o-visualization $DIVERSITY/evenness-group-significance.qzv\n",
    "module unload qiime2/2018.2;"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### PERMANOVA to sample composition in the context of categorical metadata \n",
    "Analyze sample composition in the context of categorical metadata using PERMANOVA (first described in Anderson (2001)) using the beta-group-significance command. The following commands will test whether distances between samples within a group, such as samples from the same body site (e.g., gut), are more similar to each other then they are to samples from the other groups (e.g., tongue, left palm, and right palm). \n",
    "\n",
    "If you call this command with the --p-pairwise parameter, as we’ll do here, it will also perform pairwise tests that will allow you to determine which specific pairs of groups (e.g., tongue and gut) differ from one another, if any. This command can be slow to run, especially when passing --p-pairwise, since it is based on permutation tests. So, unlike the previous commands, we’ll run this on specific columns of metadata that we’re interested in exploring, rather than all metadata columns that it’s applicable to. Here we’ll apply this to our unweighted UniFrac distances, using two sample metadata columns, as follows."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 79,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: /workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/070.diversity/unweighted-unifrac-DHA_mgkg-group-significance.qzv\u001b[0m\n",
      "\u001b[32mSaved Visualization to: /workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/070.diversity/unweighted-unifrac-Treatment-group-significance.qzv\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "#Assess a single category from the metadata file using this code OR several categories individually with the following code. \n",
    "# This DID work for both but required DHA_mgkg to be made categorical. I changed names to low Med High. Check Seasonal template\n",
    "\n",
    "module load qiime2/2018.2;\n",
    "qiime diversity beta-group-significance \\\n",
    "  --i-distance-matrix $DIVERSITY/unweighted_unifrac_distance_matrix.qza \\\n",
    "  --m-metadata-file $WORKING/metadata5dManuka.tsv \\\n",
    "  --m-metadata-column DHA_mgkg \\\n",
    "  --o-visualization $DIVERSITY/unweighted-unifrac-DHA_mgkg-group-significance.qzv \\\n",
    "  --p-pairwise\n",
    "\n",
    "qiime diversity beta-group-significance \\\n",
    "  --i-distance-matrix $DIVERSITY/unweighted_unifrac_distance_matrix.qza \\\n",
    "  --m-metadata-file $WORKING/metadata5dManuka.tsv \\\n",
    "  --m-metadata-column Treatment \\\n",
    "  --o-visualization $DIVERSITY/unweighted-unifrac-Treatment-group-significance.qzv \\\n",
    "  --p-pairwise\n",
    "  module unload qiime2/2018.2;"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 80,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Job <325086> is submitted to default queue <normal>.\n",
      "Job <325087> is submitted to default queue <normal>.\n",
      "Job <325088> is submitted to default queue <normal>.\n",
      "Job <325089> is submitted to default queue <normal>.\n"
     ]
    }
   ],
   "source": [
    "#Assess several categories \n",
    "# DID NOT WORK. WHY?\n",
    "\n",
    "module load qiime2/2018.2;\n",
    "\n",
    "for METADATACATEGORY in Treatment DHA_mgkg MG_mgkg TreatmentDHA_mgkg\n",
    "do\n",
    "bsub \\\n",
    "-J QIIME2 \\\n",
    "-o $LOG/${METADATACATEGORY}_beta_group_significance.out \\\n",
    "-e $LOG/${METADATACATEGORY}_beta_group_significance.err \\\n",
    "\"qiime diversity beta-group-significance \\\n",
    "  --i-distance-matrix $DIVERSITY/unweighted_unifrac_distance_matrix.qza \\\n",
    "  --m-metadata-file $WORKING/metadata5dManuka.tsv \\\n",
    "  --m-metadata-category ${METADATACATEGORY} \\\n",
    "  --o-visualization $DIVERSITY/${METADATACATEGORY}_unweighted-unifrac-group-significance.qzv \\\n",
    "  --p-pairwise\"\n",
    "done;\n",
    "module unload qiime2/2018.2;\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 97,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Job <325086> is not found\n"
     ]
    },
    {
     "ename": "",
     "evalue": "255",
     "output_type": "error",
     "traceback": []
    }
   ],
   "source": [
    "bjobs 325086"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The above code works fine for the metadata categories that are categorical.\n",
    "\n",
    "This method cannot operate on a grouping vector with only unique values (e.g., there are no 'within' distances because each group of objects contains only a single object).\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### ORDINATION to explore composition of microbial communtities\n",
    "\n",
    "Ordination is a popular approach for exploring microbial community composition in the context of sample metadata. \n",
    "Use the Emperor tool to explore principal coordinates (PCoA) plots in the context of sample metadata. \n",
    "An optional parameter, --p-custom-axes, is useful for exploring time series data.\n",
    "\n",
    "The PCoA results that were used in core-metrics-phylogeny are also available, making it easy to generate new visualizations with Emperor. We will generate Emperor plots for unweighted UniFrac and Bray-Curtis so that the resulting plot will contain axes for principal coordinate 1, principal coordinate 2, and days since the experiment start. We will use that last axis to explore how these samples changed over time."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 101,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: /workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/070.diversity/OrdinationUnweighted-unifrac-emperor.qzv\u001b[0m\n",
      "\u001b[32mSaved Visualization to: /workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/070.diversity/OrdinationBray-curtis-emperor.qzv\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "#AS THIS trial does NOT have TIME SERIES DATA the --p-custom-axes chunk DOES NOT NEED TO BE CONDUCTED\n",
    "#  --p-custom-axes DaysSinceExperimentStart \\\n",
    "module load qiime2/2018.2;\n",
    "qiime emperor plot \\\n",
    "  --i-pcoa $DIVERSITY/unweighted_unifrac_pcoa_results.qza \\\n",
    "  --m-metadata-file $WORKING/metadata5dManuka.tsv \\\n",
    "  --o-visualization $DIVERSITY/OrdinationUnweighted-unifrac-emperor.qzv\n",
    "\n",
    "qiime emperor plot \\\n",
    "  --i-pcoa $DIVERSITY/bray_curtis_pcoa_results.qza \\\n",
    "  --m-metadata-file $WORKING/metadata5dManuka.tsv \\\n",
    "  --o-visualization $DIVERSITY/OrdinationBray-curtis-emperor.qzv\n",
    "  module unload qiime2/2018.2;"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# 10. Taxonomic classification¶"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#Obtain greengenes classifier data.. commented out since we are not using greengenes\n",
    "#wget -O \"$WORKING/gg-13-8-99-515-806-nb-classifier.qza\" \"https://data.qiime2.org/2017.7/common/gg-13-8-99-515-806-nb-classifier.qza\""
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "###  Classify features according to the supplied taxonomy: SILVA "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Job <248191> is submitted to default queue <normal>.\n",
      "Job <248192> is submitted to default queue <normal>.\n",
      "Job <248193> is submitted to default queue <normal>.\n",
      "Job <248194> is submitted to default queue <normal>.\n",
      "Job <248195> is submitted to default queue <normal>.\n",
      "Job <248196> is submitted to default queue <normal>.\n",
      "Job <248197> is submitted to default queue <normal>.\n",
      "Job <248198> is submitted to default queue <normal>.\n"
     ]
    }
   ],
   "source": [
    "#DO NOT USE THIS ONE\n",
    "\n",
    "#Obtain SILVA classifier data\n",
    "#wget -O \"$WORKING/silva-132-99-nb-classifier.qza\" \n",
    "\n",
    "#Classify features according to the supplied taxonomy: SILVA \n",
    "# to be open licensed in July 2018\n",
    "SILVA=\"/workspace/ComparativeDataSources/SILVA/silva-132/silva-132-99-nb-classifier.qza\"\n",
    "\n",
    "mkdir -p $WORKING/logs;\n",
    "mkdir -p $WORKING/tmp;\n",
    "TMPDIR=$WORKING/tmp\n",
    "\n",
    "export TMPDIR=$TMPDIR\n",
    "\n",
    "module unload qiime2/2018.2;\n",
    "module load qiime2/2018.2;\n",
    "\n",
    "for i in 0 0.1 0.2 0.3 0.4 0.5 0.6 0.7\n",
    "do\n",
    "COMMAND=\"qiime feature-classifier classify-sklearn \\\n",
    "  --i-classifier $SILVA \\\n",
    "  --i-reads $FILTER/rep-seqs-dada2.qza \\\n",
    "  --p-confidence $i \\\n",
    "  --verbose \\\n",
    "  --o-classification $TAXONOMY/taxonomy_silva${i}.qza; \\\n",
    "qiime metadata tabulate \\\n",
    "  --m-input-file $TAXONOMY/taxonomy_silva${i}.qza \\\n",
    "  --o-visualization $TAXONOMY/taxonomy_silva${i}.qzv; \\\n",
    "qiime taxa barplot \\\n",
    "  --i-table $FILTER/table-dada2.qza \\\n",
    "  --i-taxonomy $TAXONOMY/taxonomy_silva${i}.qza \\\n",
    "  --m-metadata-file $WORKING/metadata5dManuka.tsv \\\n",
    "  --o-visualization $TAXONOMY/taxa-bar-plots_silva${i}.qzv;\"\n",
    "\n",
    "bsub -J Classify -o $WORKING/logs/%J_classify.out -e $WORKING/logs/%J_classify.err \"$COMMAND\"\n",
    "\n",
    "#echo $COMMAND\n",
    "done\n",
    "\n",
    "module unload qiime2/2018.2;\n",
    "module unload qiime2"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "###  Classify features according to the supplied taxonomy: CustomBLAST "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Job <373335> is submitted to default queue <normal>.\n",
      "Job <373336> is submitted to default queue <normal>.\n",
      "Job <373337> is submitted to default queue <normal>.\n",
      "Job <373338> is submitted to default queue <normal>.\n",
      "Job <373339> is submitted to default queue <normal>.\n"
     ]
    }
   ],
   "source": [
    "# USE THIS ONE \n",
    "#Classify features according to the supplied taxonomy: CUSTOM\n",
    "\n",
    "CUSTOMDATABASE=\"/workspace/Hrrmat/ApisMeliferraCustomBLAST/Amelliferataxonomy_feature_classifierPB150818.qza\"\n",
    "mkdir -p $WORKING/logs;\n",
    "mkdir -p $WORKING/tmp;\n",
    "TMPDIR=$WORKING/tmp\n",
    "\n",
    "export TMPDIR=$TMPDIR\n",
    "\n",
    "module unload qiime2/2018.2;\n",
    "module load qiime2/2018.2;\n",
    "\n",
    "for i in 0 0.1 0.3 0.5 0.7\n",
    "do\n",
    "COMMAND=\"qiime feature-classifier classify-sklearn \\\n",
    "  --i-classifier $CUSTOMDATABASE \\\n",
    "  --i-reads $FILTER/rep-seqs-dada2.qza \\\n",
    "  --p-confidence $i \\\n",
    "  --verbose \\\n",
    "  --o-classification $TAXONOMY/taxonomy_custom_150818${i}.qza; \\\n",
    "qiime metadata tabulate \\\n",
    "  --m-input-file $TAXONOMY/taxonomy_custom_150818${i}.qza \\\n",
    "  --o-visualization $TAXONOMY/taxonomy_custom_150818${i}.qzv; \\\n",
    "qiime taxa barplot \\\n",
    "  --i-table $FILTER/table-dada2.qza \\\n",
    "  --i-taxonomy $TAXONOMY/taxonomy_custom_150818${i}.qza \\\n",
    "  --m-metadata-file $WORKING/metadata5dManuka.tsv \\\n",
    "  --o-visualization $TAXONOMY/taxa-bar-plots_custom_150818${i}.qzv;\"\n",
    "\n",
    "bsub -J Classify  -o $WORKING/logs/%J_classify.out -e $WORKING/logs/%J_classify.err \"$COMMAND\"\n",
    "\n",
    "#echo $COMMAND\n",
    "done\n",
    "\n",
    "module unload qiime2/2018.2;\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "JOBID   USER    STAT  QUEUE      FROM_HOST   EXEC_HOST   JOB_NAME   SUBMIT_TIME\n",
      "373335  hrrmat  DONE  normal     aklppr31    wkoppb32    Classify   Aug 15 13:56\n"
     ]
    }
   ],
   "source": [
    "bjobs 373335"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "http://storage.powerplant.pfr.co.nz/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/080.taxonomy\n"
     ]
    }
   ],
   "source": [
    "#We create a link to download the tree file\n",
    "echo http://storage.powerplant.pfr.co.nz$TAXONOMY"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# 11. Differential abundance testing with ANCOM\n",
    "ANCOM can be applied to identify features that are differentially abundant (i.e. present in different abundances) across sample groups. As with any bioinformatics method, you should be aware of the assumptions and limitations of ANCOM before using it. We recommend reviewing the ANCOM paper before using this method.\n",
    "\n",
    "Note: Differential abundance testing in microbiome analysis is an active area of research. There are two QIIME 2 plugins that can be used for this: q2-gneiss and q2-composition. This section uses q2-composition, but there is another tutorial which uses gneiss on a different dataset if you are interested in learning more.\n",
    "\n",
    "ANCOM is implemented in the q2-composition plugin. ANCOM assumes that few (less than about 25%) of the features are changing between groups. If you expect that more features are changing between your groups, you should not use ANCOM as it will be more error-prone (an increase in both Type I and Type II errors is possible). Because we expect a lot of features to change in abundance across body sites, in this tutorial we’ll filter our full feature table to only contain gut samples. We’ll then apply ANCOM to determine which, if any, sequence variants and genera are differentially abundant across the gut samples of our two subjects.\n",
    "\n",
    "We’ll start by creating a feature table that contains only the gut samples. (To learn more about filtering, see the Filtering Data tutorial.)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: /workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/080.taxonomy/gut-table.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "#NOt required as this dataset only has honey bee gut samples. The variation may be >25% so even using gneiss may not be good. \n",
    "module load qiime2/2018.2;\n",
    "\n",
    "qiime feature-table filter-samples \\\n",
    "  --i-table $FILTER/table-dada2.qza \\\n",
    "  --m-metadata-file $WORKING/metadata5dManuka.tsv \\\n",
    "  --p-where \"Treatment='gut'\" \\\n",
    "  --o-filtered-table $TAXONOMY/gut-table.qza\n",
    "  module unload qiime2/2018.2;"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Composition] to: /workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/080.taxonomy/ANCOM-comp-table.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "#ANCOM operates on a FeatureTable[Composition] QIIME 2 artifact, \n",
    "#This is based on frequencies of features on a per-sample basis, but cannot tolerate frequencies of zero.\n",
    "#To build the composition artifact, a FeatureTable[Frequency] artifact must be provided to add-pseudocount (an imputation method)\n",
    "#This produces the FeatureTable[Composition] artifact.\n",
    "module load qiime2/2018.2;\n",
    " qiime composition add-pseudocount \\\n",
    "  --i-table $FILTER/table-dada2.qza \\\n",
    "  --o-composition-table $TAXONOMY/ANCOM-comp-table.qza\n",
    " module unload qiime2/2018.2;"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: /workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/080.taxonomy/Ancom-Treatment.qzv\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "# Run ANCOM on the treatment column to determine what features differ in abundance across the gut samples of the seven subjects.\n",
    "#I DO NOT know how to interpret this.\n",
    "\n",
    "module load qiime2/2018.2;\n",
    "qiime composition ancom \\\n",
    "  --i-table $TAXONOMY/ANCOM-comp-table.qza \\\n",
    "  --m-metadata-file $WORKING/metadata5dManuka.tsv \\\n",
    "  --m-metadata-column Treatment \\\n",
    "  --o-visualization $TAXONOMY/Ancom-Treatment.qzv\n",
    " module unload qiime2/2018.2;"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Differential abundance test at specified taxonomic level\n",
    "We’re also often interested in performing a differential abundance test at a specific taxonomic level. To do this, we can collapse the features in our FeatureTable[Frequency] at the taxonomic level of interest, and then re-run the above steps. Below we collapse our feature table at the genus level (i.e. level 6 of the Greengenes taxonomy)."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Usage: qiime taxa collapse [OPTIONS]\n",
      "\n",
      "  Collapse groups of features that have the same taxonomic assignment\n",
      "  through the specified level. The frequencies of all features will be\n",
      "  summed when they are collapsed.\n",
      "\n",
      "Options:\n",
      "  --i-table ARTIFACT PATH FeatureTable[Frequency]\n",
      "                                  Feature table to be collapsed.  [required]\n",
      "  --i-taxonomy ARTIFACT PATH FeatureData[Taxonomy]\n",
      "                                  Taxonomic annotations for features in the\n",
      "                                  provided feature table. All features in the\n",
      "                                  feature table must have a corresponding\n",
      "                                  taxonomic annotation. Taxonomic annotations\n",
      "                                  that are not present in the feature table\n",
      "                                  will be ignored.  [required]\n",
      "  --p-level INTEGER               The taxonomic level at which the features\n",
      "                                  should be collapsed. All ouput features will\n",
      "                                  have exactly this many levels of taxonomic\n",
      "                                  annotation.  [required]\n",
      "  --o-collapsed-table ARTIFACT PATH FeatureTable[Frequency]\n",
      "                                  The resulting feature table, where all\n",
      "                                  features are now taxonomic annotations with\n",
      "                                  the user-specified number of levels.\n",
      "                                  [required if not passing --output-dir]\n",
      "  --output-dir DIRECTORY          Output unspecified results to a directory\n",
      "  --cmd-config PATH               Use config file for command options\n",
      "  --verbose                       Display verbose output to stdout and/or\n",
      "                                  stderr during execution of this action.\n",
      "                                  [default: False]\n",
      "  --quiet                         Silence output if execution is successful\n",
      "                                  (silence is golden).  [default: False]\n",
      "  --help                          Show this message and exit.\n"
     ]
    }
   ],
   "source": [
    "module load qiime2/2018.2;\n",
    " qiime taxa collapse --help\n",
    "module unload qiime2/2018.2;"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: /workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/080.taxonomy/Collapsed-table-dada2.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "#check notes: https://docs.qiime2.org/2018.6/tutorials/moving-pictures/?highlight=unweighted%20unifrac\n",
    "module load qiime2/2018.2;\n",
    " qiime taxa collapse \\\n",
    "  --i-table $FILTER/table-dada2.qza \\\n",
    "  --i-taxonomy $TAXONOMY/taxonomy_custom_1508180.7.qza \\\n",
    "  --p-level 6 \\\n",
    "  --o-collapsed-table $TAXONOMY/Collapsed-table-dada2.qza\n",
    "\n",
    " module unload qiime2/2018.2;"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Composition] to: /workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/080.taxonomy/comp-table-dada2.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "module load qiime2/2018.2;\n",
    "qiime composition add-pseudocount \\\n",
    "  --i-table $TAXONOMY/Collapsed-table-dada2.qza \\\n",
    "  --o-composition-table $TAXONOMY/comp-table-dada2.qza\n",
    "\n",
    "module unload qiime2/2018.2;"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 56,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: /workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/080.taxonomy/Ancom-Treatment.qzv\u001b[0m\n"
     ]
    }
   ],
   "source": [
    " module load qiime2/2018.2;\n",
    "qiime composition ancom \\\n",
    "  --i-table $TAXONOMY/comp-table-dada2.qza \\\n",
    "  --m-metadata-file $WORKING/metadata5dManuka.tsv \\\n",
    "  --m-metadata-column Treatment \\\n",
    "  --o-visualization $TAXONOMY/Ancom-Treatment.qzv\n",
    " module unload qiime2/2018.2;"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Differential abundance analysis with GNEISS\n",
    "#### https://docs.qiime2.org/2018.6/tutorials/gneiss/\n",
    "#### https://forum.qiime2.org/t/gneiss-tutorial/932\n",
    "\n",
    "Perform differential abundance analysis using balances in gneiss.\n",
    "The focus is how to identify differentially abundant taxa in a compositionally coherent way.\n",
    "\n",
    "Compositionality refers to the issue of dealing with proportions. Since the sequencing depth is not typically biologically relevant, the microbial abundances are best interpreted as proportions. Because of this, it becomes challenging to infer exactly which microbes are changing – since proportions add to one, the change of a single microbe will also change the proportions of the remaining microbes.\n",
    "\n",
    "Consider the following example: ../../_images/gneiss-bars.jpg\n",
    "On the left, are the true abundances of ten species, and the first species doubles between Time point 1 and Time point 2. When we normalize these to proportions, it appears as if all of the species have changed between the two time points. Looking at proportions alone, we would never realize this problem, and we actually cannot exactly determine which species are changing based on proportions alone.\n",
    "\n",
    "While we cannot exactly solve the problem of identifying differentially abundant species, we can relax this problem and ask which partitions of microbes are changing. In the case above, if we compute the ratio between the first species and the second species, that ratio will be 1:1 at Time point 1, and 2:1 at Time point 2 for both the original abundances and the proportions. This is the type of question that balances try to solve. Rather than focusing on individual taxa, we can focus on the log ratios, since the log ratios are consist between the original abundances and the observed proportions of the species are observed. The concept of calculating balances for multiple species can be extended to trees as shown in the following example.\n",
    "\n",
    "On the left, we define a tree, where each of the tips corresponds to a taxon and their corresponding proportions. The internal nodes define the log ratio between the taxa underneath. On the right, we have the same tree, but one of the taxa abundances changes. As we have observed before, the proportions of all of the taxa will change, but looking at the balances, only the balances containing the purple taxa will change. In this case, balance b3 won’t change, since it only considers the ratio between the red and green taxa. By looking at balances instead proportions, we can control for the variance by restricting observations to only focus on the taxa within the balance.\n",
    "\n",
    "The outstanding question here is, how do we construct a tree to control for the variation, and identify interesting differentially abundant partitions of taxa? In gneiss, there are two main ways that this can be done\n",
    "\n",
    "Gradient clustering. For example, if we want to evaluate if pH is a driving factor, we can group together according to the pH that they are observed in, and observe whether the ratios of low-pH organisms to high-pH organisms change as the pH changes. This is available in the gradient-clustering command.\n",
    "Correlation clustering. If we don’t have relevant prior information about how to cluster together organisms, we can group together organisms based on how often they co-occur with each other.\n",
    "Once we have a tree, we can calculate balances using the following equation\n",
    "\n",
    "bi=rsr+s−−−−−√logg(xr)g(xs)\n",
    "where i represents the ith internal node in the tree, g(x) represents the geometric mean within set x, and xr represents the set of taxa abundances in the numerator of the balance, xs represents the set of taxa abundances in the denominator of the balance, and r and s represents the number of taxa within xr and xs respectively.\n",
    "\n",
    "After the balances are calculated, standard statistical procedures such as ANOVA and linear regression can be performed. We will see how we can run these procedures using the Chronic Fatigue Syndrome dataset.\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Building linear models using balances\n",
    "\n",
    "The differential abundance techniques that we will be running will utilize log ratio transforms. Since it is not possible to take the logarithm of zero, we will be adding in a pseudocount to all of the counts via the add-pseudocount method. In this way, we can apply logarithms on this transformed table"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Composition] to: /workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/080.taxonomy/composition.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "module load qiime2/2018.2;\n",
    "qiime gneiss add-pseudocount \\\n",
    "    --i-table $FILTER/table-dada2.qza \\\n",
    "    --p-pseudocount 1 \\\n",
    "    --o-composition-table $DIFF/composition.qza;\n",
    "module unload qiime2/2018.2;"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Define partitions of microbes for which we want to construct balances.\n",
    "Where x and y represent the proportions of two microbes across all of the samples. If two microbes are highly correlated, then this quantity will shrink close to zero. Ward hierarchical cluster will then use this distance metric to iteratively cluster together groups of microbes that are correlated with each other. In the end, the tree that we obtain will highlight the high level structure and identify the blocks within in the data."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Hierarchy to: /workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/080.taxonomy/hierarchy.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "# Produce a tree that defines the partitions\n",
    "module load qiime2/2018.2;\n",
    "qiime gneiss correlation-clustering \\\n",
    "    --i-table $DIFF/composition.qza \\\n",
    "    --o-clustering $DIFF/hierarchy.qza;\n",
    "module unload qiime2/2018.2;"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Balance] to: /workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/080.taxonomy/balances.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "# Perform the ILR transform.\n",
    "# The ILR transform computes the log ratios between groups; \n",
    "# In this case it computes log ratios between groups of anti-correlated features\n",
    "module load qiime2/2018.2;\n",
    "qiime gneiss ilr-transform \\\n",
    "    --i-table $DIFF/composition.qza \\\n",
    "    --i-tree $DIFF/hierarchy.qza \\\n",
    "    --o-balances $DIFF/balances.qza\n",
    "module unload qiime2/2018.2;"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Once we have obtained a means to partition the features, we can now run linear regression on the balances. The abundances will be converted to principal balances using the partition scheme that we defined earlier. The linear regression that we will be running is called a multivariate response linear regression, which performs a linear regression on each balance separately.\n",
    "\n",
    "We can use this to attempt to predict the microbial abundances based on environmental variables. Running these models has multiple advantages over standard univariate regression, as it avoids many of the issues associated with overfitting, and can gain perspective about community-wide perturbations based on environmental parameters.\n",
    "\n",
    "Since the microbial abundances can be mapped directly to balances, we can perform this multivariate response directly on the balances. The model that we will be building is represented as follows.\n",
    "y⃗ =β0→+βSubject→Xsubject→+βsex→Xsex→+βage→XAge→+βsCD14ugml→XsCD14ugml→+βLBPugml→XLBPugml→\n",
    "\n",
    "Where y⃗  represents the matrix of balances to be predicted, βi→ represents a vector of coefficients for covariate i and math:vec{X_i} represents the measures for covariate i.\n",
    "\n",
    "ANOVA is a special case of linear regression - every problem that can be solved by ANOVA can be reformulated as a linear regression. For more details see https://stats.stackexchange.com/questions/76250/r-anova-and-linear-regression/76292#76292. So we can still answer the same sort of differential abundance questions using this technique, but we can start asking more precise questions, controlling for different potential confounding variables or even interaction effects."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: /workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/080.taxonomy/regression_summary.qzv\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "# Run a Multivariate Response linear regression on the balances. \n",
    "#This performs a linear regression on each balance separately.\n",
    "TMPDIR=$WORKING/tmp\n",
    "export TMPDIR=$TMPDIR;\n",
    "module load qiime2/2018.2;\n",
    "for METADATACATEGORY in Treatment DHA_mgkg MG_mgkg TreatmentDHA_mgkg\n",
    "do\n",
    "qiime gneiss ols-regression \\\n",
    "    --p-formula \"${METADATACATEGORY}\" \\\n",
    "    --i-table $DIFF/balances.qza \\\n",
    "    --i-tree $DIFF/hierarchy.qza \\\n",
    "    --m-metadata-file $WORKING/metadata5dManuka.tsv \\\n",
    "    --o-visualization $DIFF/${METADATACATEGORY}_regression_summary.qzv;\n",
    "done\n",
    "module unload qiime2/2018.2;"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Now we have a summary of the regression model. Specifically we want to see which covariates impact the model the most, which balances are meaningful, and how much potential overfitting is going on.\n",
    "\n",
    "There are a few things to note in the regression summary. There is an R2 in the summary, which gives information about the variance in the community is explained by the regression model. From what we can see, the regression can explain about 10% of the community. This is typical for what we see in human gut microbes, since there is a very high amount of confounding variation due to genetics, diet, environment, etc.\n",
    "\n",
    "To evaluate the explanatory model of a single covariate, a leave-one-variable-out approach is used. One variable is left out, and the change in R2 is calculated. The larger the change is, the stronger the effect of the covariate is. In this case, Subject is the largest explanatory factor, explaining 2% of the variation.\n",
    "\n",
    "To make sure that we aren’t overfitting, 10-fold cross validation is performed. This will split the data into 10 partitions, build the model on 9 of the those partitions and use the remaining partition to measure the prediction accuracy. This process is repeat 10 times, once for each round of cross-validation. The within model error (mse), R2 and the prediction accuracy (pred_mse) are reported for each round of cross validation. Here, the prediction accuracy is less than the within model error, suggesting that over fitting is not happening.\n",
    "\n",
    "Next, we have a heatmap visualizing all of the coefficient p-values for all of the balances. The columns of the heatmap represent balances, and the rows of the heatmap represent covariates. The heatmap is colored by the negative log of the p-value, highlighting potentially significant p-values. A hover tool is enabled to allow for specific coefficient values and their corresponding p-values to be obtained, and zooming is enabled to allow for navigation of interesting covariates and balances.\n",
    "\n",
    "Next are the prediction and residual plots. Here, only the top two balances are plotted, and the prediction residuals from the model are projected onto these two balances. From these plots we can see that the predicted points lie within the same region as the original communities. However, we can see that the residuals have roughly the same variance as the predictions. This is a little unsettling - but note that we can only explain 10% of the community variance, so these sorts of calculations aren’t completely unexpected.\n",
    "\n",
    "The branch lengths in the visualized tree are also scaled by the explained sum of squares in the models. The longest branch lengths correspond to the most informative balances. This can allow us to get a high-level overview of the most important balances in the model. From this plot and the above heatmap, we can see that balance y0 is important. These balances not only have very small p-values (with p<0.05) for differentiating subjects, but they also have the largest branch lengths in the tree diagram. This suggests that these two partitions of microbes could differentiate the CFS patients from the controls.\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: /workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/080.taxonomy/heatmapBalances.qzv\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "# Visualize these balances on a heatmap to see which groups of OTUs they represent. \n",
    "# By default, the values within the feature table are log-scaled, with the sample means centered around zero.\n",
    "module load qiime2/2018.2;\n",
    "for METADATACOLUMN in Treatment Cage\n",
    "do\n",
    "qiime gneiss dendrogram-heatmap \\\n",
    "    --i-table $DIFF/composition.qza \\\n",
    "    --i-tree $DIFF/hierarchy.qza \\\n",
    "    --m-metadata-file $WORKING/metadata5dManuka.tsv \\\n",
    "    --m-metadata-column ${METADATACOLUMN} \\\n",
    "    --p-color-map seismic \\\n",
    "    --o-visualization $DIFF/${METADATACOLUMN}_heatmap.qzv;\n",
    "    done\n",
    "module unload qiime2/2018.2;\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Usage: qiime gneiss dendrogram-heatmap [OPTIONS]\n",
      "\n",
      "  Visualize the feature table as a heatmap, with samples sorted along a\n",
      "  specified categorical metadata column and features clustered together\n",
      "  specified by the tree.\n",
      "\n",
      "Options:\n",
      "  --i-table ARTIFACT PATH FeatureTable[Composition]\n",
      "                                  The feature table that will be plotted as a\n",
      "                                  heatmap. This table is assumed to have\n",
      "                                  strictly positive values.  [required]\n",
      "  --i-tree ARTIFACT PATH Hierarchy\n",
      "                                  A hierarchy of feature identifiers where\n",
      "                                  each tipcorresponds to the feature\n",
      "                                  identifiers in the table. This tree can\n",
      "                                  contain tip ids that are not present in the\n",
      "                                  table, but all feature ids in the table must\n",
      "                                  be present in this tree.  [required]\n",
      "  --m-metadata-file MULTIPLE PATH\n",
      "                                  Metadata file or artifact viewable as\n",
      "                                  metadata. This option may be supplied\n",
      "                                  multiple times to merge metadata.\n",
      "                                  [required]\n",
      "  --m-metadata-column MetadataColumn[Categorical]\n",
      "                                  Column from metadata file or artifact\n",
      "                                  viewable as metadata. Categorical metadata\n",
      "                                  column to group the samples.  [required]\n",
      "  --p-ndim INTEGER                Number of dimensions to highlight.\n",
      "                                  [default: 10]\n",
      "  --p-method [log|clr]            Specifies how the data should be normalized\n",
      "                                  for display.Options include 'log' or 'clr'\n",
      "                                  (default='clr').  [default: clr]\n",
      "  --p-color-map [Vega10|Purples|summer|cool|gist_earth|YlOrBr|PuOr|RdYlBu|PiYG|gist_stern|OrRd|YlGn|Vega20b|gist_ncar|Pastel2|magma|terrain|CMRmap|Accent|BrBG|plasma|hsv|BuGn|PuBu|rainbow|GnBu|Set3|YlOrRd|RdBu|gist_heat|winter|Set1|gray|flag|PRGn|Greens|afmhot|ocean|Paired|hot|viridis|coolwarm|inferno|Dark2|brg|prism|gnuplot2|PuRd|Blues|bone|Greys|Oranges|Vega20|cubehelix|pink|bwr|nipy_spectral|gist_rainbow|copper|spring|RdYlGn|Pastel1|autumn|Vega20c|Spectral|gnuplot|Set2|YlGnBu|seismic|Reds|RdGy|jet|PuBuGn|RdPu|BuPu]\n",
      "                                  Specifies the color map for plotting the\n",
      "                                  heatmap. See https://matplotlib.org/examples\n",
      "                                  /color/colormaps_reference.html for more\n",
      "                                  details.  [default: viridis]\n",
      "  --o-visualization VISUALIZATION PATH\n",
      "                                  [required if not passing --output-dir]\n",
      "  --output-dir DIRECTORY          Output unspecified results to a directory\n",
      "  --cmd-config PATH               Use config file for command options\n",
      "  --verbose                       Display verbose output to stdout and/or\n",
      "                                  stderr during execution of this action.\n",
      "                                  [default: False]\n",
      "  --quiet                         Silence output if execution is successful\n",
      "                                  (silence is golden).  [default: False]\n",
      "  --help                          Show this message and exit.\n"
     ]
    }
   ],
   "source": [
    "module load qiime2/2018.2;\n",
    "qiime gneiss dendrogram-heatmap --help\n",
    "module unload qiime2/2018.2;"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "As noted in the legend, the numerators for each balance are highlighted in light red, while the denominators are highlighted in dark red. From here, we can see that the denominator of y0 has few OTUs compared to the numerator of y0. These taxa in the denominator could be interesting, so let’s investigate the taxonomies making up this balance with balance_taxonomy.\n",
    "\n",
    "Specifically we’ll plot a boxplot and identify taxa that could be explaining the differences between the control and patient groups."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 57,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\r\n",
      "Aborted!\r\n"
     ]
    }
   ],
   "source": [
    "# Produce boxplot and identify taxa to explain differences between treatments. Use Taxa-level 6 as 7 reads to subspecies\n",
    "module load qiime2/2018.2;\n",
    "qiime gneiss balance-taxonomy \\\n",
    "  --i-table $DIFF/composition.qza \\\n",
    "  --i-tree $DIFF/hierarchy.qza \\\n",
    "  --i-taxonomy $TAXONOMY/taxonomy_custom_1508180.7.qza \\\n",
    "  --p-taxa-level 6 \\\n",
    "  --p-balance-name 'y0' \\\n",
    "  --m-metadata-file $WORKING/metadata5dManuka.tsv \\\n",
    "  --m-metadata-column Treatment \\\n",
    "  --o-visualization $DIFF/y0_taxa_summaryLevel6.qzv\n",
    "module unload qiime2/2018.2;"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: /workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/090.differential_abundance/Treatment_taxa_summary.qzv\u001b[0m\n",
      "\u001b[32mSaved Visualization to: /workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/090.differential_abundance/DHA_mgkg_taxa_summary.qzv\u001b[0m\n",
      "\u001b[32mSaved Visualization to: /workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/090.differential_abundance/MG_mgkg_taxa_summary.qzv\u001b[0m\n",
      "\u001b[32mSaved Visualization to: /workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/090.differential_abundance/HMF_mgkg_taxa_summary.qzv\u001b[0m\n",
      "\u001b[32mSaved Visualization to: /workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/090.differential_abundance/NPA_taxa_summary.qzv\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "### This shows number of unique taxa. \n",
    "#Loop code for above to show number of unique taxa. \n",
    "#Adjust --p-taxa-level to see species levels.\n",
    "#Adjust --i-taxonomy $TAXONOMY/taxonomy_silva0.7.qza file number to see classification levels\n",
    "\n",
    "module load qiime2/2018.2;\n",
    "for METADATACOLUMN in Treatment DHA_mgkg MG_mgkg HMF_mgkg NPA\n",
    "do\n",
    "qiime gneiss balance-taxonomy \\\n",
    "    --i-table $DIFF/composition.qza \\\n",
    "    --i-tree $DIFF/hierarchy.qza \\\n",
    "    --i-taxonomy $TAXONOMY/taxonomy_custom_1508180.7.qza \\\n",
    "    --p-taxa-level 6 \\\n",
    "    --p-balance-name 'y0' \\\n",
    "    --m-metadata-file $WORKING/metadata5dManuka.tsv \\\n",
    "    --m-metadata-column ${METADATACOLUMN} \\\n",
    "    --o-visualization $DIFF/${METADATACOLUMN}_taxa_summary.qzv;\n",
    "done\n",
    "module unload qiime2/2018.2;"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "In this particular case, the log ratio is lower in the patient group compared to the control group. In essence, this means that the taxa in the y0numerator on average are more abundant than the taxa in y0denominator in the healthy control group compared to the patient group.\n",
    "\n",
    "Remember, based on the toy examples given in the beginning of this tutorial, it is not possible to infer absolute changes of microbes in a given sample. Balances will not be able to provide this sort of answer, but it can limit the number of possible scenarios. Specifically, one of the five following scenarios could have happened.\n",
    "\n",
    "The taxa in the y0numerator on average have increased between patient group and the healthy control.\n",
    "The taxa in the y0denominator on average have decreased between patient group and the healthy control.\n",
    "A combination of the above occurred\n",
    "Taxa abundances in both y0numerator and y0denominator both increased, but taxa abundances in y0numerator increased more compared to y0denominator\n",
    "Taxa abundances in both y0numerator and y0denominator both decreased, but taxa abundances in y0denominator increased more compared to y0numerator\n",
    "To further narrow down these hypothesis, biological prior knowledge or experimental validation will be required."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "module load qiime2/2018.2;\n",
    "\n",
    "module unload qiime2/2018.2;"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# 12. Produce a biom file called feature-table.biom (composition.qza) and a dna-sequences.fasta file"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Creating a BIOM table with taxonomy annotations 27Sept2018"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#Look here for help\n",
    "# https://forum.qiime2.org/t/exporting-and-modifying-biom-tables-e-g-adding-taxonomy-annotations/3630"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 60,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "# Create a Biom Table with taxonomy annotations \n",
    "#27Sept2018 With Patrick Biggs\n",
    "#Export table.qza which for me is table-dada2.qza and it becomes the file \"feature-table.biom\"\n",
    "#Export taxonomy.qza which for me is taxonomy_custom_1508180.7.qza and it becomes the file \"taxonomy.tsv\"\n",
    "module load qiime2/2018.2;\n",
    "qiime tools export /powerplant/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/040.filter/table-dada2.qza --output-dir $BIOM_TABLE \n",
    "qiime tools export /powerplant/workspace/Hrrmat/QIIME2/MGS7R1_5dManuka/080.taxonomy/taxonomy_custom_1508180.7.qza --output-dir $BIOM_TABLE \n",
    "module unload qiime2/2018.2;\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 61,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "# Make a copy of the exported taxonomy file ($BIOM_TABLE/taxonomy.tsv) \n",
    " \n",
    "cp $BIOM_TABLE/taxonomy.tsv $BIOM_TABLE/biom-taxonomy.tsv\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#Modify the header before using it with BIOM software.\n",
    "#Manualy change the first line of biom-taxonomy.tsv (i.e. the header) to this:\n",
    "# #OTUID\ttaxonomy\tconfidence\n",
    "#Note that you’ll need to use tab characters in the header since this is a TSV file.\n",
    "\n",
    "#Your biom-taxonomy.tsv file should now look something like this (the actual data will differ):\n",
    "# \"#OTUID\ttaxonomy\tconfidence\"\n",
    "#0204aa97a655fab087a6f6902be35eb6\tk__Bacteria; p__Bacteroidetes; c__Sphingobacteriia; o__Sphingobacteriales; f__Sphingobacteriaceae; g__Sphingobacterium; s__faecium\t-1.0\n",
    "#5cfeeb662c643e512cbfc724c984f53d\tk__Bacteria; p__Firmicutes; c__Bacilli; o__Lactobacillales; f__Lactobacillaceae; g__Lactobacillus; s__\t-1.0\n",
    "#4dc10ad4afd35b9803d853e819d3cec5\tk__Bacteria; p__Firmicutes; c__Clostridia; o__Clostridiales; f__Lachnospiraceae; g__; s__\t-1.0\n",
    "#...\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 71,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#Finally, add the taxonomy data to the biom file: THIS ADDs THE TAXONOMY\n",
    "#http://biom-format.org/documentation/adding_metadata.html\n",
    "module load qiime2/2018.2;\n",
    "biom add-metadata -i $BIOM_TABLE/feature-table.biom -o $BIOM_TABLE/table-with-taxonomy.biom --observation-metadata-fp $BIOM_TABLE/biom-taxonomy.tsv --sc-separated taxonomy\n",
    "\n",
    "#Then check\n",
    "biom convert -i $BIOM_TABLE/table-with-taxonomy.biom -o $BIOM_TABLE/table-with-taxonomy.tsv --to-tsv --header-key taxonomy\n",
    "module unload qiime2/2018.2;\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Next step, load feature_table.tsv into excel and save as CSV file for use in R phyloseq."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 58,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Usage: biom add-metadata [OPTIONS]\n",
      "\n",
      "  Add metadata to a BIOM table.\n",
      "\n",
      "  Add sample and/or observation metadata to BIOM-formatted files. See\n",
      "  examples here: http://biom-format.org/documentation/adding_metadata.html\n",
      "\n",
      "  Example usage:\n",
      "\n",
      "  Add sample metadata to a BIOM table:\n",
      "\n",
      "  $ biom add-metadata -i otu_table.biom -o table_with_sample_metadata.biom\n",
      "  -m sample_metadata.txt\n",
      "\n",
      "Options:\n",
      "  -i, --input-fp PATH             The input BIOM table  [required]\n",
      "  -o, --output-fp PATH            The output BIOM table  [required]\n",
      "  -m, --sample-metadata-fp PATH   The sample metadata mapping file (will add\n",
      "                                  sample metadata to the input BIOM table, if\n",
      "                                  provided).\n",
      "  --observation-metadata-fp PATH  The observation metadata mapping file (will\n",
      "                                  add observation metadata to the input BIOM\n",
      "                                  table, if provided).\n",
      "  --sc-separated TEXT             Comma-separated list of the metadata fields\n",
      "                                  to split on semicolons. This is useful for\n",
      "                                  hierarchical data such as taxonomy or\n",
      "                                  functional categories.\n",
      "  --sc-pipe-separated TEXT        Comma-separated list of the metadata fields\n",
      "                                  to split on semicolons and pipes (\"|\"). This\n",
      "                                  is useful for hierarchical data such as\n",
      "                                  functional categories with one-to-many\n",
      "                                  mappings (e.g. x;y;z|x;y;w)).\n",
      "  --int-fields TEXT               Comma-separated list of the metadata fields\n",
      "                                  to cast to integers. This is useful for\n",
      "                                  integer data such as \"DaysSinceStart\".\n",
      "  --float-fields TEXT             Comma-separated list of the metadata fields\n",
      "                                  to cast to floating point numbers. This is\n",
      "                                  useful for real number data such as \"pH\".\n",
      "  --sample-header TEXT            Comma-separated list of the sample metadata\n",
      "                                  field names. This is useful if a header line\n",
      "                                  is not provided with the metadata, if you\n",
      "                                  want to rename the fields, or if you want to\n",
      "                                  include only the first n fields where n is\n",
      "                                  the number of entries provided here.\n",
      "  --observation-header TEXT       Comma-separated list of the observation\n",
      "                                  metadata field names. This is useful if a\n",
      "                                  header line is not provided with the\n",
      "                                  metadata, if you want to rename the fields,\n",
      "                                  or if you want to include only the first n\n",
      "                                  fields where n is the number of entries\n",
      "                                  provided here.\n",
      "  --output-as-json                Write the output file in JSON format.\n",
      "  -h, --help                      Show this message and exit.\n"
     ]
    }
   ],
   "source": [
    "#If help is required then use this code:\n",
    "module load qiime2/2018.2;\n",
    "biom add-metadata -h\n",
    "module unload qiime2/2018.2;"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": true
   },
   "source": [
    "#### Save notebook to Github using the following address\n",
    "##### https://jupyterhub.powerplant.pfr.co.nz/user/hrrmat/notebooks/Hrrmat/Github-Creating%20and%20updating%20a%20Jupyter%20notebook%20.ipynb"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#Provides help list of functions for gneiss\n",
    "module load qiime2/2018.2;\n",
    "qiime gneiss balance-taxonomy;\n",
    "module unload qiime2/2018.2;"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "ls $TAXONOMY"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Bash",
   "language": "bash",
   "name": "bash"
  },
  "language_info": {
   "codemirror_mode": "shell",
   "file_extension": ".sh",
   "mimetype": "text/x-sh",
   "name": "bash"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
